<!doctype html>
<html lang="en">

<head>

    


    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- CSRF Token -->
    <meta name="csrf-token" content="R3qucnRi4lsggouZbx2aFXEvUs8OyUXAcnFSpYMq">
        <title>Upcoming Courses - SIB Training</title>

    <link href="https://www.sib.swiss/training2/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />

    <!-- Styles -->
    <link href="https://www.sib.swiss/training2/css/website.css?id=1195888e0f7b6876a61fbeac0b576d9e" rel="stylesheet">

        <title>Upcoming training courses</title>
<meta name="description" content="The SIB Swiss Institute of Bioinformatics is an academic not-for-profit organization whose mission is to lead and coordinate the field of bioinformatics in Switzerland.">

    <meta property="og:title" content="Upcoming training courses" />
<meta property="og:description" content="The SIB Swiss Institute of Bioinformatics is an academic not-for-profit organization whose mission is to lead and coordinate the field of bioinformatics in Switzerland." />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://www.sib.swiss/training2/website/upcoming-training-courses" />
<meta property="og:image" content="https://www.sib.swiss/images/sib/4-training/training_header_2018.jpg" />


    <meta name="twitter:site" content="@ISBSIB" />
<meta name="twitter:image" content="https://www.sib.swiss/images/sib/4-training/training_header_2018.jpg" />
<meta name="twitter:title" content="Upcoming training courses" />
<meta name="twitter:description" content="The SIB Swiss Institute of Bioinformatics is an academic not-for-profit organization whose mission is to lead and coordinate the field of bioinformatics in Switzerland." />

    <script type="application/ld+json">[{"@context":"https://schema.org","@type":"Course","name":"Advanced Statistics: Statistical Modelling","description":"\n**This course is now full with a long waiting list. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our [upcoming courses](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) (if not yet done). Thank you for your understanding.** \n# Overview\nWhile the statistical models and tools presented in an introductory statistics course (such as linear regression) can be used to answer a wide range of questions in life sciences, many types of data cannot be analyzed using these simple approaches.\n\nDuring this course, we will discuss statistical models and techniques beyond classical linear modelling. Following a brief review of the basics of linear regression, we will dive into more advanced topics, such as generalized and mixed-effects linear models. We will further discuss the application of mixed-effects linear models in analyzing longitudinal data. Finally, in an attempt to move beyond linearity, we will explore extensions of linear models, such as polynomial regression, splines, local regression, and generalized additive models. Throughout the course, the emphasis will be put on concrete applications in clinical and biological data analysis using real world examples.\n\n# Audience\nThis course is intended for life scientists who already use the R programming language and have some basic knowledge of statistics (including statistical tests, correlation, and linear models).\n\n# Learning outcomes\nAt the end of this course, participants will be able to:\n*  identify the appropriate model to analyze a dataset;\n*  fit the chosen model using R;\n*  assess the fit of the model, as well as its limitations.\n\n# Prerequisites\n##### Knowledge / competencies\nThe course is intended for people already **familiar with basic statistics and R**. Participants must be comfortable with topics such as hypothesis testing, correlation and linear models, and must have a **prior knowledge of the \"R\" language and environment for statistical computing and graphics**. Participants who have already followed the SIB course [\"Introduction to statistics with R\"](https://www.sib.swiss/training/course/2021-02-intro-stats) or an equivalent course, and have used its content in practice should fit this prerequisite.  \n\n**Before applying to this course, please self-assess your knowledge in stats and R to make sure this course is right for you. Here are 2 quizzes:**  \n[https://gohighbrow.com/quiz-introduction-to-statistics/](https://gohighbrow.com/quiz-introduction-to-statistics/)  \n[https://docs.google.com/forms/d/e/1FAIpQLSfXCnmLha0Ks4ZZZ42G_5MyIbGi-JhPayuHZ_P2jdXZEtXdqg/viewform](https://docs.google.com/forms/d/e/1FAIpQLSfXCnmLha0Ks4ZZZ42G_5MyIbGi-JhPayuHZ_P2jdXZEtXdqg/viewform)\n\n\n##### Technical\nYou are required to have **your own laptop, with at least 4 Gb of RAM, [\"R\" v4.2.0](https://www.r-project.org/) and [\"RStudio\" 2022.02.2-485](https://www.rstudio.com/products/rstudio/download/#download) software installed**. More information about the packages needed will be provided in due time. \n\n# Brief course programme\n*  Monday: simple and multiple linear regression (theory, diagnostics, and model selection)\n*  Tuesday: generalized linear models (binary data, proportions, and counts)\n*  Wednesday: mixed-effects linear models, longitudinal data analysis\n*  Thursday:smoothing and generalized additive models\n\n# Application\n\n\nThe registration fees for academics are **240 CHF** and **1200 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **15/08/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to this and other general conditions, available [here](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be held in Lausanne. \n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Monique Zahn, SIB training group.\n\nWe will recommend 1 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 4-day course presents statistical models and techniques beyond classical linear modelling, with an emphasis on concrete applications in biology.","educationalLevel":"Advanced","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Advanced Statistics: Statistical Modelling","description":"\n**This course is now full with a long waiting list. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our [upcoming courses](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) (if not yet done). Thank you for your understanding.** \n# Overview\nWhile the statistical models and tools presented in an introductory statistics course (such as linear regression) can be used to answer a wide range of questions in life sciences, many types of data cannot be analyzed using these simple approaches.\n\nDuring this course, we will discuss statistical models and techniques beyond classical linear modelling. Following a brief review of the basics of linear regression, we will dive into more advanced topics, such as generalized and mixed-effects linear models. We will further discuss the application of mixed-effects linear models in analyzing longitudinal data. Finally, in an attempt to move beyond linearity, we will explore extensions of linear models, such as polynomial regression, splines, local regression, and generalized additive models. Throughout the course, the emphasis will be put on concrete applications in clinical and biological data analysis using real world examples.\n\n# Audience\nThis course is intended for life scientists who already use the R programming language and have some basic knowledge of statistics (including statistical tests, correlation, and linear models).\n\n# Learning outcomes\nAt the end of this course, participants will be able to:\n*  identify the appropriate model to analyze a dataset;\n*  fit the chosen model using R;\n*  assess the fit of the model, as well as its limitations.\n\n# Prerequisites\n##### Knowledge / competencies\nThe course is intended for people already **familiar with basic statistics and R**. Participants must be comfortable with topics such as hypothesis testing, correlation and linear models, and must have a **prior knowledge of the \"R\" language and environment for statistical computing and graphics**. Participants who have already followed the SIB course [\"Introduction to statistics with R\"](https://www.sib.swiss/training/course/2021-02-intro-stats) or an equivalent course, and have used its content in practice should fit this prerequisite.  \n\n**Before applying to this course, please self-assess your knowledge in stats and R to make sure this course is right for you. Here are 2 quizzes:**  \n[https://gohighbrow.com/quiz-introduction-to-statistics/](https://gohighbrow.com/quiz-introduction-to-statistics/)  \n[https://docs.google.com/forms/d/e/1FAIpQLSfXCnmLha0Ks4ZZZ42G_5MyIbGi-JhPayuHZ_P2jdXZEtXdqg/viewform](https://docs.google.com/forms/d/e/1FAIpQLSfXCnmLha0Ks4ZZZ42G_5MyIbGi-JhPayuHZ_P2jdXZEtXdqg/viewform)\n\n\n##### Technical\nYou are required to have **your own laptop, with at least 4 Gb of RAM, [\"R\" v4.2.0](https://www.r-project.org/) and [\"RStudio\" 2022.02.2-485](https://www.rstudio.com/products/rstudio/download/#download) software installed**. More information about the packages needed will be provided in due time. \n\n# Brief course programme\n*  Monday: simple and multiple linear regression (theory, diagnostics, and model selection)\n*  Tuesday: generalized linear models (binary data, proportions, and counts)\n*  Wednesday: mixed-effects linear models, longitudinal data analysis\n*  Thursday:smoothing and generalized additive models\n\n# Application\n\n\nThe registration fees for academics are **240 CHF** and **1200 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **15/08/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to this and other general conditions, available [here](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be held in Lausanne. \n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Monique Zahn, SIB training group.\n\nWe will recommend 1 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220822_XXXX3","@id":"https://www.sib.swiss/training2/website/course/20220822_XXXX3","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Advanced Statistics: Statistical Modelling"}],"courseMode":["onsite","synchronous"],"keywords":"training,biostatistics,raphael gottardo group","location":"Lausanne","endDate":"2022-08-25","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Mauro Delorenzi"},{"@type":"Person","name":"Rachel Jeitziner Marcone"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":240,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":1200,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-08-22","duration":"P3DT4H"}]},{"@context":"https://schema.org","@type":"Course","name":"Best Practices in Programming","description":"# Overview\nThe workshop will focus on learning and internalizing the practices of unit testing, refactoring, and version control through hands-on experience. The first morning will start with an introduction into these concepts and tools used to support them. In the afternoon, we will transition to a *code clinic* and work together in small groups applying these practices to make improvements to **code brought by participants**. The second day will continue with the code clinic.\n\nThe focus of this course is not object-oriented programming, software architecture, design patterns or algorithms. The goal of this course is to introduce skills and techniques for effectively developing software.\n\n# Audience\nIf the answer to two or more of the following questions is “yes”, then this course is for you.\n\nDo you write software to analyze data or implement in-silico models as part of your daily work?\nDo you write scientific publications based on software you created yourself but do not know what unit testing, refactoring or version control are?\nAre you writing code that you find hard to understand some weeks later?\nDo you have thousands of lines of code, but no automated way of verifying that the code works correctly?\nDo you find yourself regularly using “copy & paste” to re-use code you wrote earlier?\nDo small changes in your code later cause trouble in other unexpected places?\n\n# Learning objectives\nAt the end of the workshop, the participants should be able to:\n\nidentify and avoid the most common mistakes in the process of writing software in a scientific context\nimprove the quality of her/his code\nexploit techniques for effectively developing software\n\n# Prerequisites\n##### Knowledge / competencies\n* Working knowledge of one of the following programming languages: R, python, C/C++, java, perl, Matlab.\n\n##### Technical / material\n* Participants will need send some extracts of their code at least 2 weeks before the course\n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **20/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be take in Zurich and start at 9:00 and end around 17:00. Detailed information will be provided to the participants.\n\n\n#  Additional information\nCoordination: Grégoire Rossier\n\nInstructors: Uwe Schmitt, Manuel Weberndorfer, Swen Vermeul, Gerhard Bräunlich, Maria Del, John Hennig (all IT Services, ETHZ).\n\nWe will recommend 0.5 ECTS credits for this course (given that you provide your code and work on it).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":null,"educationalLevel":"Intermediate","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Best Practices in Programming","description":"# Overview\nThe workshop will focus on learning and internalizing the practices of unit testing, refactoring, and version control through hands-on experience. The first morning will start with an introduction into these concepts and tools used to support them. In the afternoon, we will transition to a *code clinic* and work together in small groups applying these practices to make improvements to **code brought by participants**. The second day will continue with the code clinic.\n\nThe focus of this course is not object-oriented programming, software architecture, design patterns or algorithms. The goal of this course is to introduce skills and techniques for effectively developing software.\n\n# Audience\nIf the answer to two or more of the following questions is “yes”, then this course is for you.\n\nDo you write software to analyze data or implement in-silico models as part of your daily work?\nDo you write scientific publications based on software you created yourself but do not know what unit testing, refactoring or version control are?\nAre you writing code that you find hard to understand some weeks later?\nDo you have thousands of lines of code, but no automated way of verifying that the code works correctly?\nDo you find yourself regularly using “copy & paste” to re-use code you wrote earlier?\nDo small changes in your code later cause trouble in other unexpected places?\n\n# Learning objectives\nAt the end of the workshop, the participants should be able to:\n\nidentify and avoid the most common mistakes in the process of writing software in a scientific context\nimprove the quality of her/his code\nexploit techniques for effectively developing software\n\n# Prerequisites\n##### Knowledge / competencies\n* Working knowledge of one of the following programming languages: R, python, C/C++, java, perl, Matlab.\n\n##### Technical / material\n* Participants will need send some extracts of their code at least 2 weeks before the course\n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **20/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be take in Zurich and start at 9:00 and end around 17:00. Detailed information will be provided to the participants.\n\n\n#  Additional information\nCoordination: Grégoire Rossier\n\nInstructors: Uwe Schmitt, Manuel Weberndorfer, Swen Vermeul, Gerhard Bräunlich, Maria Del, John Hennig (all IT Services, ETHZ).\n\nWe will recommend 0.5 ECTS credits for this course (given that you provide your code and work on it).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220706_XXXX4","@id":"https://www.sib.swiss/training2/website/course/20220706_XXXX4","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Best Practices in Programming"}],"courseMode":["onsite","synchronous"],"keywords":"programming,reproducibility,bernd rinn group","location":"Zurich","endDate":"2022-07-07","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Manuel Weberndorfer"},{"@type":"Person","name":"Swen Vermeul"},{"@type":"Person","name":"Gerhard Bräunlich"},{"@type":"Person","name":"Maria Del"},{"@type":"Person","name":"Uwe Schmitt"},{"@type":"Person","name":"John Hennnig"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-07-06","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"Docker and Singularity for reproducible research: getting started with containers","description":"**This course is now full.** You can still apply, in which case your application will be put onto the waiting list and you will be recontacted IF and ONLY IF a seat becomes available. Thank you for your understanding.  \n\n\n# Overview\nReproducible analysis in bioinformatics goes beyond good documentation and sharing code. Often, analyses depend on an entire environment with many different programs, libraries and settings. For the sake of reproducibility, but also for productivity, you might want to store, reuse, and share your environments using container software such as Docker and Singularity. Containerization is a lightweight and scalable way of working with such isolated environments. For example, with Docker you can develop, store and manage environments based on Linux operating systems which can run in mostly any computer. \n\nOften, bioinformatic calculations also require large memory and CPU resources, therefore you might want to use a shared computer environment (e.g. a HPC cluster). However, to run Docker you need to have privileged rights (i.e. as root), which is not the case for Singularity, a program that enables you to run environments from a wide range of resources (including Docker) in a safe manner without the need for privileged access. Combining Singularity and Docker thus makes you extremely flexible in how, when and where you use your computer environments and those of others. \n\nTo introduce you to working with containers in the context of bioinformatics research, we will take a practical approach with several hands-on exercises to go through the basics of using, developing and sharing containers.\n\nThe course materials can be found in the dedicated [GitHub page](https://sib-swiss.github.io/containers-introduction-training/latest/).\n# Audience\nThis course is addressed to bioinformaticians and life scientists that are doing bioinformatic analyses and want to get introduced to containers.\n\n# Learning outcomes\nAt the end of the course, the participants are expected to:\n* Understand the basic concepts and terminology associated with virtualization with containers\n* Customize, store, manage and share containerized environments with Docker\n* Describe and understand the essential differences between Docker and Singularity\n* Use Singularity to run containers on a shared computer environment (e.g. a HPC cluster)\n\n# Prerequisites\n##### Knowledge / competencies\nThis course is designed for those interested to work with containers in the context of bioinformatics. Practical experience on the basic UNIX commands is required. You can test your skills with Unix with the quiz [here](https://docs.google.com/forms/d/e/1FAIpQLSd2BEWeOKLbIRGBT_aDEGPce1FOaVYBbhBiaqcaHoBKNB27MQ/viewform?usp=sf_link). If you do not feel comfortable with UNIX commands, you can refresh your knowledge with our [Unix fundamentals](https://edu.sib.swiss/course/view.php?name=UNIX_I) e-learning module.\n\n##### Technical\nYou are required to bring your own laptop with [Docker pre-installed](https://docs.docker.com/get-docker/), and have created an account on [dockerhub](https://hub.docker.com/). Note that you need admin rights in your computer to install and use Docker. In addition, Windows users require a recent version of [Windows](https://docs.docker.com/docker-for-windows/install/). \n\n# Schedule \nThe schedule and course material are in the dedicated [GitHub page]( https://sib-swiss.github.io/containers-introduction-training/latest/course_schedule/\n ).\n# Application\n**This course is now full.** You can still apply, in which case your application will be put onto the waiting list and you will be recontacted IF and ONLY IF a seat becomes available. Thank you for your understanding.  \n\n\n\nSIB courses usually open for applications 1-2 months before the course takes place. If not yet done, I suggest you sign up for our mailing list [here](https://lists.sib.swiss/mailman/listinfo/courses) so that you are informed when the courses accept applications as some courses fill up very quickly. \n\nRegistration fees for academics are **60 CHF** and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **12 October 2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will only be streamed. \n\n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"In this course, you will see how Singularity and Docker can help make bioinformatics analysis reproducible, and go beyond good documentation and code shari","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Docker and Singularity for reproducible research: getting started with containers","description":"**This course is now full.** You can still apply, in which case your application will be put onto the waiting list and you will be recontacted IF and ONLY IF a seat becomes available. Thank you for your understanding.  \n\n\n# Overview\nReproducible analysis in bioinformatics goes beyond good documentation and sharing code. Often, analyses depend on an entire environment with many different programs, libraries and settings. For the sake of reproducibility, but also for productivity, you might want to store, reuse, and share your environments using container software such as Docker and Singularity. Containerization is a lightweight and scalable way of working with such isolated environments. For example, with Docker you can develop, store and manage environments based on Linux operating systems which can run in mostly any computer. \n\nOften, bioinformatic calculations also require large memory and CPU resources, therefore you might want to use a shared computer environment (e.g. a HPC cluster). However, to run Docker you need to have privileged rights (i.e. as root), which is not the case for Singularity, a program that enables you to run environments from a wide range of resources (including Docker) in a safe manner without the need for privileged access. Combining Singularity and Docker thus makes you extremely flexible in how, when and where you use your computer environments and those of others. \n\nTo introduce you to working with containers in the context of bioinformatics research, we will take a practical approach with several hands-on exercises to go through the basics of using, developing and sharing containers.\n\nThe course materials can be found in the dedicated [GitHub page](https://sib-swiss.github.io/containers-introduction-training/latest/).\n# Audience\nThis course is addressed to bioinformaticians and life scientists that are doing bioinformatic analyses and want to get introduced to containers.\n\n# Learning outcomes\nAt the end of the course, the participants are expected to:\n* Understand the basic concepts and terminology associated with virtualization with containers\n* Customize, store, manage and share containerized environments with Docker\n* Describe and understand the essential differences between Docker and Singularity\n* Use Singularity to run containers on a shared computer environment (e.g. a HPC cluster)\n\n# Prerequisites\n##### Knowledge / competencies\nThis course is designed for those interested to work with containers in the context of bioinformatics. Practical experience on the basic UNIX commands is required. You can test your skills with Unix with the quiz [here](https://docs.google.com/forms/d/e/1FAIpQLSd2BEWeOKLbIRGBT_aDEGPce1FOaVYBbhBiaqcaHoBKNB27MQ/viewform?usp=sf_link). If you do not feel comfortable with UNIX commands, you can refresh your knowledge with our [Unix fundamentals](https://edu.sib.swiss/course/view.php?name=UNIX_I) e-learning module.\n\n##### Technical\nYou are required to bring your own laptop with [Docker pre-installed](https://docs.docker.com/get-docker/), and have created an account on [dockerhub](https://hub.docker.com/). Note that you need admin rights in your computer to install and use Docker. In addition, Windows users require a recent version of [Windows](https://docs.docker.com/docker-for-windows/install/). \n\n# Schedule \nThe schedule and course material are in the dedicated [GitHub page]( https://sib-swiss.github.io/containers-introduction-training/latest/course_schedule/\n ).\n# Application\n**This course is now full.** You can still apply, in which case your application will be put onto the waiting list and you will be recontacted IF and ONLY IF a seat becomes available. Thank you for your understanding.  \n\n\n\nSIB courses usually open for applications 1-2 months before the course takes place. If not yet done, I suggest you sign up for our mailing list [here](https://lists.sib.swiss/mailman/listinfo/courses) so that you are informed when the courses accept applications as some courses fill up very quickly. \n\nRegistration fees for academics are **60 CHF** and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **12 October 2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will only be streamed. \n\n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221026_XXXX8","@id":"https://www.sib.swiss/training2/website/course/20221026_XXXX8","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Docker and Singularity for reproducible research: getting started with containers"}],"courseMode":["onsite","synchronous"],"keywords":"reproducibility,training,container,data management,data security,high-performance computing,rémy bruggmann group","location":"Streamed","endDate":"2022-10-26","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Geert van Geest"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-26"}]},{"@context":"https://schema.org","@type":"Course","name":"Enrichment Analysis","description":"**This course is now full with a long waiting list. If you do not want to miss your chance to be part of the next session, we highly recommend you keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) (if not yet done).**\n\n# Overview\nExperiments designed to quantify gene expression often yield hundreds of genes that show statistically significant differences between two classes (two biological states, two phenotype states, two experimental conditions, etc). Once differentially expressed genes are identified, enrichment analysis (EA) methods can be conducted to identify groups of genes (e.g. particular pathways) that are differentially expressed, and offer insights into biological mechanisms. One example of such a method is the Gene Set Enrichment Analysis (GSEA), which is very popular and frequently used for high-throughput gene expression data analysis.\n\nThis course will cover GSEA and alternative enrichment methods. Because most of their implementations are directly linked to databases of functional annotation of genes in the cell, the course will also give an overview of functional annotation databases such as the Gene Ontology resource.\n\n\n# Audience\nBiologists eager to identify a statistically reliable set of genes that are differentially expressed.\n\n# Learning objectives\nAt the end of the course, the participants will be able to:\n*  Distinguish available enrichment analysis methods\n*  Apply GSEA and over-representation analysis using R\n*  Determine whether the genes of a Gene Ontology term have a statistically significant difference in expression or not\n*  Learn where to find other gene sets in databases (e.g. KEGG, oncogenic gene sets) and use them in R.   \n\n\n# Prerequisites\n*Knowledge / Competencies*\n* statistics beginner level (T-test, multiple testing methods). \n* R beginner level (Rstudio, install a library, matrix manipulation, read files). [Test your R skills with the quiz here, before registering.](https://docs.google.com/forms/d/e/1FAIpQLSdIyeuabd_ZOWXgI1MWHapmaOMu20L9ESkLDZiWnpmkpujyOg/viewform?usp=sf_link) \n\n\n*Technical*\n* This course will be streamed, you are thus required to have your own computer with an internet connection, and with latest [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/products/rstudio/download/) versions installed. \n\n# Application\nRegistration fees are **60 CHF** for academics and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **17/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n # Venue and Time\nThis course will be streamed.  \n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants on due time.\n#  Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 1 day course will cover Gene Set Enrichment Analysis to identify groups of genes, alternative enrichment methods, as well as the Gene Ontology resourc","educationalLevel":"Intermediate","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Enrichment Analysis","description":"**This course is now full with a long waiting list. If you do not want to miss your chance to be part of the next session, we highly recommend you keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) (if not yet done).**\n\n# Overview\nExperiments designed to quantify gene expression often yield hundreds of genes that show statistically significant differences between two classes (two biological states, two phenotype states, two experimental conditions, etc). Once differentially expressed genes are identified, enrichment analysis (EA) methods can be conducted to identify groups of genes (e.g. particular pathways) that are differentially expressed, and offer insights into biological mechanisms. One example of such a method is the Gene Set Enrichment Analysis (GSEA), which is very popular and frequently used for high-throughput gene expression data analysis.\n\nThis course will cover GSEA and alternative enrichment methods. Because most of their implementations are directly linked to databases of functional annotation of genes in the cell, the course will also give an overview of functional annotation databases such as the Gene Ontology resource.\n\n\n# Audience\nBiologists eager to identify a statistically reliable set of genes that are differentially expressed.\n\n# Learning objectives\nAt the end of the course, the participants will be able to:\n*  Distinguish available enrichment analysis methods\n*  Apply GSEA and over-representation analysis using R\n*  Determine whether the genes of a Gene Ontology term have a statistically significant difference in expression or not\n*  Learn where to find other gene sets in databases (e.g. KEGG, oncogenic gene sets) and use them in R.   \n\n\n# Prerequisites\n*Knowledge / Competencies*\n* statistics beginner level (T-test, multiple testing methods). \n* R beginner level (Rstudio, install a library, matrix manipulation, read files). [Test your R skills with the quiz here, before registering.](https://docs.google.com/forms/d/e/1FAIpQLSdIyeuabd_ZOWXgI1MWHapmaOMu20L9ESkLDZiWnpmkpujyOg/viewform?usp=sf_link) \n\n\n*Technical*\n* This course will be streamed, you are thus required to have your own computer with an internet connection, and with latest [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/products/rstudio/download/) versions installed. \n\n# Application\nRegistration fees are **60 CHF** for academics and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **17/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n # Venue and Time\nThis course will be streamed.  \n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants on due time.\n#  Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220624_XXXX9","@id":"https://www.sib.swiss/training2/website/course/20220624_XXXX9","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Enrichment Analysis"}],"courseMode":["onsite","synchronous"],"keywords":"biostatistics,experimental biology,functional genomics,genes and genomes,training,raphael gottardo group","location":"Streamed","endDate":"2022-06-24","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Tania Wyss Lozano"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-06-24","duration":"PT7H"},{"@type":"CourseInstance","name":"Enrichment Analysis","description":"\n\n# Overview\nExperiments designed to quantify gene expression often yield hundreds of genes that show statistically significant differences between two classes (two biological states, two phenotype states, two experimental conditions, etc). Once differentially expressed genes are identified, enrichment analysis (EA) methods can be conducted to identify groups of genes (e.g. particular pathways) that are differentially expressed, and offer insights into biological mechanisms. One example of such a method is the Gene Set Enrichment Analysis (GSEA), which is very popular and frequently used for high-throughput gene expression data analysis.\n\nThis course will cover GSEA and alternative enrichment tools. Since most of their implementations are directly linked to databases that annotate the function of genes in the cell, the course will also introduce GO enrichment analysis.\n\n\n# Audience\nBiologists eager to identify a statistically reliable set of genes that are differentially expressed.\n\n# Learning objectives\nAt the end of the course, the participants will be able to:\n*  identify statistical methods that could be used to pinpoint differentially expressed genes quantified by methods such as microarrays or RNA sequencing\n*  determine whether a set of genes shows statistically significant differences between two classes or not\n*  distinguish available enrichment analysis methods\n*  apply GSEA using R\n*  apply enrichment analysis implementations using R\n*  determine whether the genes of a GO term have a statistically significant difference in expression or not\n*  learn where to find other gene sets in databases (e.g. KEGG, oncogenic gene sets) and use them in R.   \n\n\n# Prerequisites\n*Knowledge / Competencies*\n* statistics beginner level (T-test, multiple testing methods). \n* R beginner level (Rstudio, install a library, matrix manipulation, read files). [Test your R skills with the quiz here, before registering.](https://docs.google.com/forms/d/e/1FAIpQLSdIyeuabd_ZOWXgI1MWHapmaOMu20L9ESkLDZiWnpmkpujyOg/viewform?usp=sf_link) \n\n\n*Technical*\n* This course will be streamed, you are thus required to have your own computer with an internet connection, and with latest [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/products/rstudio/download/) versions installed. An online access to R will also be provided for the practical exercises. \n\n# Application\nRegistration fees are **60 CHF** for academics and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **16/11/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n # Venue and Time\nThis course will be streamed.  \n\nThe course will start at 9:00 CET and end around 17:00 CET.\n\nPrecise information will be provided to the participants on due time.\n#  Additional information\nCoordination: Diana Marek, SIB Training Group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221202_XXXX9","@id":"https://www.sib.swiss/training2/website/course/20221202_XXXX9","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Enrichment Analysis"}],"courseMode":["onsite","synchronous"],"keywords":"biostatistics,experimental biology,functional genomics,genes and genomes,training,mauro delorenzi & frédéric schütz group","location":"Streamed","endDate":"2022-12-02","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Tania Wyss Lozano"},{"@type":"Person","name":"Gustavo Ruiz Buendia"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-12-02","duration":"PT7H"}]},{"@context":"https://schema.org","@type":"Course","name":"First Steps in Parallelization with OpenMP","description":"**Attendance is free-of-charge, however registration is mandatory.**\n\n# Overview\nWe currently live in the era of multicore computers, where almost every computing machine has many CPUs. In order to benefit from this computational power, we need to devise programs explicitly written for parallel machines. Nowadays there are several paradigms to write parallel code, and in this crash course we will focus on [OpenMP](https://www.openmp.org/). This is a minimally invasive open-source parallelization method that allows, in many cases, fast and straight-forward parallelization of currently working serial codes. OpenMP is meant for compiled code (eg C++ or Fortran) that run in a single computing node, but we will also visit some alternatives for Python, along with a short overview of parallelization over distributed memory machines via MPI.\n\n# Audience\nThis course is addressed to researchers who want to take their first steps in parallel computing. We will assume no previous knowledge of OpenMP (nor of other parallelization techniques) from the participants.\n\n# Learning objectives\nAt the end of the course, the participants are expected to:\n\n* understand and be able to implement basic parallelization methods with OpenMP\n\n# Prerequisites\n##### Knowledge / competencies\nParticipants should be comfortable working in a Linux/UNIX environment and have some basic experience in programming. Some knowledge of C/C++ or Fortran is desirable, although not necessary.\n\n# Application\nAttendance is free-of-charge, however registration is mandatory (use the Apply button below). While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation.\n\nApplications will close as soon as the places will be filled up. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed.\n\nThe course will start at 9:00 CET and end around 17:00 CET. \n\nPrecise information will be provided to the participants before the course.\n\n\n# Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 1-day course will allow you to understand and be able to implement basic parallelization methods with OpenMP.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"First Steps in Parallelization with OpenMP","description":"**Attendance is free-of-charge, however registration is mandatory.**\n\n# Overview\nWe currently live in the era of multicore computers, where almost every computing machine has many CPUs. In order to benefit from this computational power, we need to devise programs explicitly written for parallel machines. Nowadays there are several paradigms to write parallel code, and in this crash course we will focus on [OpenMP](https://www.openmp.org/). This is a minimally invasive open-source parallelization method that allows, in many cases, fast and straight-forward parallelization of currently working serial codes. OpenMP is meant for compiled code (eg C++ or Fortran) that run in a single computing node, but we will also visit some alternatives for Python, along with a short overview of parallelization over distributed memory machines via MPI.\n\n# Audience\nThis course is addressed to researchers who want to take their first steps in parallel computing. We will assume no previous knowledge of OpenMP (nor of other parallelization techniques) from the participants.\n\n# Learning objectives\nAt the end of the course, the participants are expected to:\n\n* understand and be able to implement basic parallelization methods with OpenMP\n\n# Prerequisites\n##### Knowledge / competencies\nParticipants should be comfortable working in a Linux/UNIX environment and have some basic experience in programming. Some knowledge of C/C++ or Fortran is desirable, although not necessary.\n\n# Application\nAttendance is free-of-charge, however registration is mandatory (use the Apply button below). While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation.\n\nApplications will close as soon as the places will be filled up. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed.\n\nThe course will start at 9:00 CET and end around 17:00 CET. \n\nPrecise information will be provided to the participants before the course.\n\n\n# Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220519_XXX10","@id":"https://www.sib.swiss/training2/website/course/20220519_XXX10","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"First Steps in Parallelization with OpenMP"}],"courseMode":["onsite","synchronous"],"keywords":"API,high-performance computing,programming,training,torsten schwede & thierry sengstag group","location":"Streamed","endDate":"2022-05-19","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Rubén Cabezón"},{"@type":"Person","name":"Aurélien Cavelan"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":0,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":0,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-05-19","duration":"PT7H"},{"@type":"CourseInstance","name":"First Steps in Parallelization with OpenMP","description":"\n\n# Overview\nWe currently live in the era of multicore computers, where almost every computing machine has many CPUs. In order to benefit from this computational power, we need to devise programs explicitly written for parallel machines. Nowadays there are several paradigms to write parallel code, and in this crash course we will focus on [OpenMP](https://www.openmp.org/). This is a minimally invasive open-source parallelization method that allows, in many cases, fast and straight-forward parallelization of currently working serial codes. OpenMP is meant for compiled code (eg C++ or Fortran) that run in a single computing node, but we will also visit some alternatives for Python, along with a short overview of parallelization over distributed memory machines via MPI.\n\n# Audience\nThis course is addressed to researchers who want to take their first steps in parallel computing. We will assume no previous knowledge of OpenMP (nor of other parallelization techniques) from the participants.\n\n# Learning objectives\nAt the end of the course, the participants are expected to:\n\n* understand and be able to implement basic parallelization methods with OpenMP\n\n# Prerequisites\n##### Knowledge / competencies\nParticipants should be comfortable working in a Linux/UNIX environment and have some basic experience in programming. Some knowledge of C/C++ or Fortran is desirable, although not necessary.\n\n# Application\nAttendance is free-of-charge, however registration is mandatory (use the Apply button below). While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation.\n\nApplications will close as soon as the places will be filled up. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\n\n\nThe course will start at 9:00 CET and end around 17:00 CET. \n\nPrecise information will be provided to the participants before the course.\n\n\n# Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221123_XXX10","@id":"https://www.sib.swiss/training2/website/course/20221123_XXX10","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"First Steps in Parallelization with OpenMP"}],"courseMode":["onsite","synchronous"],"keywords":"API,high-performance computing,programming,training,torsten schwede & thierry sengstag group","location":"Streamed","endDate":"2022-11-23","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Rubén Cabezón"},{"@type":"Person","name":"Aurélien Cavelan"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":0,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":0,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-11-23","duration":"PT7H"}]},{"@context":"https://schema.org","@type":"Course","name":"First Steps with Python in Life Sciences","description":"\n\n\n\n# Overview\nHave you ever been stuck with a file format that doesn't precisely conform to your needs, found yourself doing annoyingly repetitive data manipulations, or struggled to efficiently manage and explore your data? Python to the rescue!\n\nPython is an open-source and general-purpose scripting language which runs on all major operating systems. It was designed to be easily read and written with comparatively simple syntax, and is thus a good choice for beginners in programming. Python is applied in many disciplines and is one of the most common languages for bioinformatics. The Python community enthusiastically maintains a rich collection of libraries/modules for everything from web development to machine learning. Other programming languages such as R have comparable functionality to Python, however some tasks are more natural (and easier!) in Python.\n\nIn this course, participants will learn the basic concepts and data structures necessary to solve routine data manipulation tasks. Importantly, we will cover the concepts, terminology, and approach to documentation required to further develop skills in Python programming independently. The goal is to take control of your research questions in an independent manner.\n\nTopics include:\n\n* A basic introduction to Python scripting and computing in general\n* Creating, populating, and modifying data structures\n* Working with files : reading / writing\n* Installing / Importing libraries/modules\n* Writing functions\n* Best practices in Python programming\n* Debugging and documentation\n\n# Audience\nThis 3-day course is addressed to **beginners** who want to become familiar with writing Python code to accomplish common tasks such as automated data parsing, basic statistical operations and graphical representations.\n\nFor people who are proficient in programming: this course might be on the **slow side** for you and an intermediate python class is recommended (check regularly our upcoming training courses).\n\n# Learning objectives\n\nBy the end of this course, you will not only be prepared to learn more advanced bioinformatics-specific applications in forthcoming courses, but also be able to:\n\n* Create, populate, and modify data structures\n* Work with files: read and write files\n* Install and import libraries and modules\n* Write functions\n* Apply best practices in Python programming\n* Debug and document your own code\n\n# Prerequisites\n### Knowledge / competencies\nThis course is designed for beginners; there is no requirement for previous training in Python. However, we encourage completion of our \"First Steps with UNIX\" course or our “UNIX Fundamentals” tutorial. Basic concepts of algorithmics is a plus.\n### Technical\nYou are required to have your own laptop. We will be working with Python managed by Anaconda - a free and operating system (OS)-agnostic platform for organizing Python libraries and environments. It is bundled with Anaconda Navigator, a graphical user interface which will help ease you into what Python makes possible. We will discuss in detail what all of this precisely means during the course. In preparation, all you have to do is download and install Anaconda for your particular OS. Use any most recent version of Python (>=  3.7). If your disk space is limited, you can install Miniconda instead. However this lacks the Anaconda Navigator GUI and many of the packages distributed with Anaconda.\n\nWe will also be interacting with python via the jupyter notebook interface and we ask that you install jupyter notebook, via conda.\n\n\n# Application\n\n\n\nRegistration fees are **180 CHF** for academics and **900 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **23/04/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will be held in person at the Kollegienhaus, University of Basel. \n\nIt will start at 9:00 and end around 17:00. \n\nPrecise information will be provided to the participants in due time.\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"In this course, participants will learn the basic concepts and data structures with the Python programming language to solve data manipulation tasks.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"First Steps with Python in Life Sciences","description":"\n\n\n\n# Overview\nHave you ever been stuck with a file format that doesn't precisely conform to your needs, found yourself doing annoyingly repetitive data manipulations, or struggled to efficiently manage and explore your data? Python to the rescue!\n\nPython is an open-source and general-purpose scripting language which runs on all major operating systems. It was designed to be easily read and written with comparatively simple syntax, and is thus a good choice for beginners in programming. Python is applied in many disciplines and is one of the most common languages for bioinformatics. The Python community enthusiastically maintains a rich collection of libraries/modules for everything from web development to machine learning. Other programming languages such as R have comparable functionality to Python, however some tasks are more natural (and easier!) in Python.\n\nIn this course, participants will learn the basic concepts and data structures necessary to solve routine data manipulation tasks. Importantly, we will cover the concepts, terminology, and approach to documentation required to further develop skills in Python programming independently. The goal is to take control of your research questions in an independent manner.\n\nTopics include:\n\n* A basic introduction to Python scripting and computing in general\n* Creating, populating, and modifying data structures\n* Working with files : reading / writing\n* Installing / Importing libraries/modules\n* Writing functions\n* Best practices in Python programming\n* Debugging and documentation\n\n# Audience\nThis 3-day course is addressed to **beginners** who want to become familiar with writing Python code to accomplish common tasks such as automated data parsing, basic statistical operations and graphical representations.\n\nFor people who are proficient in programming: this course might be on the **slow side** for you and an intermediate python class is recommended (check regularly our upcoming training courses).\n\n# Learning objectives\n\nBy the end of this course, you will not only be prepared to learn more advanced bioinformatics-specific applications in forthcoming courses, but also be able to:\n\n* Create, populate, and modify data structures\n* Work with files: read and write files\n* Install and import libraries and modules\n* Write functions\n* Apply best practices in Python programming\n* Debug and document your own code\n\n# Prerequisites\n### Knowledge / competencies\nThis course is designed for beginners; there is no requirement for previous training in Python. However, we encourage completion of our \"First Steps with UNIX\" course or our “UNIX Fundamentals” tutorial. Basic concepts of algorithmics is a plus.\n### Technical\nYou are required to have your own laptop. We will be working with Python managed by Anaconda - a free and operating system (OS)-agnostic platform for organizing Python libraries and environments. It is bundled with Anaconda Navigator, a graphical user interface which will help ease you into what Python makes possible. We will discuss in detail what all of this precisely means during the course. In preparation, all you have to do is download and install Anaconda for your particular OS. Use any most recent version of Python (>=  3.7). If your disk space is limited, you can install Miniconda instead. However this lacks the Anaconda Navigator GUI and many of the packages distributed with Anaconda.\n\nWe will also be interacting with python via the jupyter notebook interface and we ask that you install jupyter notebook, via conda.\n\n\n# Application\n\n\n\nRegistration fees are **180 CHF** for academics and **900 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **23/04/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will be held in person at the Kollegienhaus, University of Basel. \n\nIt will start at 9:00 and end around 17:00. \n\nPrecise information will be provided to the participants in due time.\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220504_XXX11","@id":"https://www.sib.swiss/training2/website/course/20220504_XXX11","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"First Steps with Python in Life Sciences"}],"courseMode":["onsite","synchronous"],"keywords":"programming,reproducibility,training,data management,data visualisation,torsten schwede & thierry sengstag group,mark ibberson group","location":"Basel","endDate":"2022-05-06","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"},{"@type":"Person","name":"Robin Engler and Orlin Topalov"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":180,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":900,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-05-04","duration":"P2DT5H"},{"@type":"CourseInstance","name":"First Steps with Python in Life Sciences","description":"\n\n\n\n\n\n\n# Overview\nHave you ever been stuck with a file format that doesn't precisely conform to your needs, found yourself doing annoyingly repetitive data manipulations, or struggled to efficiently manage and explore your data? Python to the rescue!\n\nPython is an open-source and general-purpose scripting language which runs on all major operating systems. It was designed to be easily read and written with comparatively simple syntax, and is thus a good choice for beginners in programming. Python is applied in many disciplines and is one of the most common languages for bioinformatics. The Python community enthusiastically maintains a rich collection of libraries/modules for everything from web development to machine learning. Other programming languages such as R have comparable functionality to Python, however some tasks are more natural (and easier!) in Python.\n\nIn this course, participants will learn the basic concepts and data structures necessary to solve routine data manipulation tasks. Importantly, we will cover the concepts, terminology, and approach to documentation required to further develop skills in Python programming independently. The goal is to take control of your research questions in an independent manner.\n\nTopics include:\n\n* A basic introduction to Python scripting and computing in general\n* Creating, populating, and modifying data structures\n* Working with files : reading / writing\n* Installing / Importing libraries/modules\n* Writing functions\n* Best practices in Python programming\n* Debugging and documentation\n\n# Audience\nThis 3-day course is addressed to **beginners** who want to become familiar with writing Python code to accomplish common tasks such as automated data parsing, basic statistical operations and graphical representations.\n\nFor people who are proficient in programming: this course might be on the **slow side** for you and an intermediate python class is recommended (check regularly our upcoming training courses).\n\n# Learning objectives\n\nBy the end of this course, you will not only be prepared to learn more advanced bioinformatics-specific applications in forthcoming courses, but also be able to:\n\n* Create, populate, and modify data structures\n* Work with files: read and write files\n* Install and import libraries and modules\n* Write functions\n* Apply best practices in Python programming\n* Debug and document your own code\n\n# Prerequisites\n### Knowledge / competencies\nThis course is designed for beginners; there is no requirement for previous training in Python. However, we encourage completion of our \"First Steps with UNIX\" course or our “UNIX Fundamentals” tutorial. Basic concepts of algorithmics is a plus.\n### Technical\nYou are required to have your own laptop. We will be working with Python managed by Anaconda - a free and operating system (OS)-agnostic platform for organizing Python libraries and environments. It is bundled with Anaconda Navigator, a graphical user interface which will help ease you into what Python makes possible. We will discuss in detail what all of this precisely means during the course. In preparation, all you have to do is download and install Anaconda for your particular OS. Use any most recent version of Python (>=  3.7). If your disk space is limited, you can install Miniconda instead. However this lacks the Anaconda Navigator GUI and many of the packages distributed with Anaconda.\n\nWe will also be interacting with python via the jupyter notebook interface and we ask that you install jupyter notebook, via conda.\n\n\n# Application\n\n\n\t\n**Application is closed.**\n\nRegistration fees are **180 CHF** for academics and **900 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **11 Sept 2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our general conditions.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will ONLY be held in person in Lausanne.\n\nIt will start at 9:00 and end around 17:00. \nPlease note that a social dinner will be organised on 28 September after the course.\n\nPrecise information will be provided to the participants in due time.\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220928_XXX11","@id":"https://www.sib.swiss/training2/website/course/20220928_XXX11","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"First Steps with Python in Life Sciences"}],"courseMode":["onsite","synchronous"],"keywords":"programming,reproducibility,training,data management,data visualisation,torsten schwede & thierry sengstag group,mark ibberson group","location":"Lausanne","endDate":"2022-09-30","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"},{"@type":"Person","name":"Robin Englern and Orlin Topalov"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":180,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":900,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-09-28","duration":"P2DT5H"}]},{"@context":"https://schema.org","@type":"Course","name":"First steps with R in Life Sciences","description":"**This course is now full.**  If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) here (if you haven't done so already).\n\n\n\n\n\n# Overview\nR is a complete, flexible and open source system for statistical analysis and graphics, which has become a tool of choice for biologists and biomedical scientists who need to analyse and visualise large amounts of data. Due to its popularity, R is continuously updated and extended with the latest libraries for data manipulation, modelling and visualisation. \n\nThis course will give an introduction to R and Rstudio (an integrated development environment for R) in an interactive manner. Participants will receive example data sets to practise data manipulation, graphical exploration and statistical hypothesis testing.\n\n# Audience\nThis \"First Steps with R\" two-day course is addressed to beginners wanting to become familiar with the R syntax, environment, and the most common commands to start using R to explore and interpret their data.\n\n# Learning objectives\nAt the end of the course, participants should be comfortable with the R environment and be able to read, understand and write R commands, allowing them to implement and interpret a workflow for their data analysis:\n* interact with the R environment\n* import data, explore it and summarise it\n* explore data with graphs\n* perform simple statistical tests\n\n\n# Prerequisites\n#### Knowledge / competencies\nAs it is a course for beginners, no background in R or any programming language is required. However, you are encouraged to go through some of the R documentation available [here](http://cran.r-project.org/doc/manuals/r-release/R-intro.html).\n\nPlease also note this course is NOT a training on statistics but rather a training on how to use R to perform different tasks, statistics being one of them.\n#### Technical\nA Wi-Fi enabled laptop with latest [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/products/rstudio/download/#download) versions installed. There will be access to the eduroam and guest-unil networks. \n\n# Application\n\n\nRegistration fees are **120 CHF** for academics and **600 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **25/05/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n\n # Venue and  time\nThis course will take place at the University of Lausanne.\n\nIt will start at 9:00 and end around 17:30.\n\nPrecise information will be provided to the participants on due time.\n\n#  Additional information\nCoordination: Valeria Di Cola.\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](http://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 2-day course will introduce participants to R syntax, the Rstudio environment, and to the use of R to explore and interpret data.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"First steps with R in Life Sciences","description":"**This course is now full.**  If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) here (if you haven't done so already).\n\n\n\n\n\n# Overview\nR is a complete, flexible and open source system for statistical analysis and graphics, which has become a tool of choice for biologists and biomedical scientists who need to analyse and visualise large amounts of data. Due to its popularity, R is continuously updated and extended with the latest libraries for data manipulation, modelling and visualisation. \n\nThis course will give an introduction to R and Rstudio (an integrated development environment for R) in an interactive manner. Participants will receive example data sets to practise data manipulation, graphical exploration and statistical hypothesis testing.\n\n# Audience\nThis \"First Steps with R\" two-day course is addressed to beginners wanting to become familiar with the R syntax, environment, and the most common commands to start using R to explore and interpret their data.\n\n# Learning objectives\nAt the end of the course, participants should be comfortable with the R environment and be able to read, understand and write R commands, allowing them to implement and interpret a workflow for their data analysis:\n* interact with the R environment\n* import data, explore it and summarise it\n* explore data with graphs\n* perform simple statistical tests\n\n\n# Prerequisites\n#### Knowledge / competencies\nAs it is a course for beginners, no background in R or any programming language is required. However, you are encouraged to go through some of the R documentation available [here](http://cran.r-project.org/doc/manuals/r-release/R-intro.html).\n\nPlease also note this course is NOT a training on statistics but rather a training on how to use R to perform different tasks, statistics being one of them.\n#### Technical\nA Wi-Fi enabled laptop with latest [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/products/rstudio/download/#download) versions installed. There will be access to the eduroam and guest-unil networks. \n\n# Application\n\n\nRegistration fees are **120 CHF** for academics and **600 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **25/05/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n\n # Venue and  time\nThis course will take place at the University of Lausanne.\n\nIt will start at 9:00 and end around 17:30.\n\nPrecise information will be provided to the participants on due time.\n\n#  Additional information\nCoordination: Valeria Di Cola.\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](http://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220608_XXX12","@id":"https://www.sib.swiss/training2/website/course/20220608_XXX12","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"First steps with R in Life Sciences"}],"courseMode":["onsite","synchronous"],"keywords":"biostatistics,data visualisation,programming","location":"Lausanne","endDate":"2022-06-09","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Leonore Wigger"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-06-08","duration":"P1DT6H"},{"@type":"CourseInstance","name":"First steps with R in Life Sciences","description":"\n\n **This course is now full with a long waiting list.**  If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses [mailing list](https://lists.sib.swiss/mailman/listinfo/courses) here (if you haven't done so already).\n\n# Overview\nR is a complete, flexible and open source system for statistical analysis and graphics, which has become a tool of choice for biologists and biomedical scientists who need to analyse and visualise large amounts of data. Due to its popularity, R is continuously updated and extended with the latest libraries for data manipulation, modelling and visualisation. \n\nThis course will give an introduction to R and Rstudio (an integrated development environment for R) in an interactive manner. Participants will receive example data sets to practise data manipulation, graphical exploration and statistical hypothesis testing.\n\n# Audience\nThis \"First Steps with R\" two-day course is addressed to beginners wanting to become familiar with the R syntax, environment, and the most common commands to start using R to explore and interpret their data.\n\n# Learning objectives\nAt the end of the course, participants should be comfortable with the R environment and be able to read, understand and write R commands, allowing them to implement and interpret a workflow for their data analysis:\n* interact with the R environment\n* import data, explore it and summarise it\n* explore data with graphs\n* perform simple statistical tests\n\n\n# Prerequisites\n#### Knowledge / competencies\nAs it is a course for beginners, no background in R or any programming language is required. However, you are encouraged to go through some of the R documentation available [here](http://cran.r-project.org/doc/manuals/r-release/R-intro.html).\n\nPlease also note this course is NOT a training on statistics but rather a training on how to use R to perform different tasks, statistics being one of them.\n#### Technical\nA Wi-Fi enabled laptop with latest [R](https://www.r-project.org/) and [RStudio](https://www.rstudio.com/products/rstudio/download/#download) versions installed. There will be access to the eduroam and guest network. \n\n# Application\n\n\nThe course is now full with a long waiting list. We therefore can't take any other application. There will be other sessions in the future. Thank you for your understanding.\n\nRegistration fees are **120 CHF** for academics and **600 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **26/09/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n\n # Venue and  time\nThis course will take place at the University of Zurich, on the Irchel campus.\n\nIt will start at 9:00 and end around 17:30.\n\nPrecise information will be provided to the participants on due time.\n\n#  Additional information\nCoordination: Diana Marek.\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](http://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221010_XXX12","@id":"https://www.sib.swiss/training2/website/course/20221010_XXX12","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"First steps with R in Life Sciences"}],"courseMode":["onsite","synchronous"],"keywords":"training,biostatistics,data visualisation,programming,torsten schwede & thierry sengstag group","location":"Zürich","endDate":"2022-10-11","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-10","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"Galaxy introduction for life scientists","description":"\n\n\n# Overview\nGalaxy is an open, web-based platform for accessible, reproducible, and transparent computational biomedical research. Galaxy gives you access to thousands of tools. For example, you can run a complete NGS analysis from fastq files to publication-ready figures using only your web browser.\n\nThis course will give a general introduction on the galaxy web-page structure, how to import data, run tools, and share analyses. Participants will run a whole NGS analysis using an RNA-seq dataset as an example.\n\n# Audience\nThis course is targeted to any life scientist who wants to learn more about the web-based [Galaxy platform](https://usegalaxy.eu/)  and would like to discover its potential to analyze NGS data.\n\n# Learning objectives\nAfter the course, participants should be able to use a public Galaxy server for their research and be able to:\n\n* reproduce and share an analysis in Galaxy\n* run tools such as FastQC, cutadapt, STAR\n* understand their outputs\n* interpret the biological results obtained with Galaxy\n\n# Prerequisites\n##### Knowledge / competencies\nNo command line experience or programming knowledge is required. However, familiarity with NGS and genomic data is highly recommended.\n\n##### Technical\nThe participants need to come with their Wi-Fi enabled laptop with Firefox, Chrome, or Safari web browser installed. There will be access to the eduroam (and guest) networks.\n\n# Application\n\n\nRegistration fees are **60 CHF** for academics and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed. \n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **20/04/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nIf the sanitary situation allows it, this course will be held in person at the University of Lausanne  or at the EPFL. Otherwise it will be live-streamed online.\n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n#  Additional information\nTrainers: Lucille Delisle (EPFL) and Hans-Rudolf Hotz (FMI, SIB)\n\nCoordination: Monique Zahn, SIB Training group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 1-day course will give a general introduction on the Galaxy web page structure, how to import data, run tools, and share analyses.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Galaxy introduction for life scientists","description":"\n\n\n# Overview\nGalaxy is an open, web-based platform for accessible, reproducible, and transparent computational biomedical research. Galaxy gives you access to thousands of tools. For example, you can run a complete NGS analysis from fastq files to publication-ready figures using only your web browser.\n\nThis course will give a general introduction on the galaxy web-page structure, how to import data, run tools, and share analyses. Participants will run a whole NGS analysis using an RNA-seq dataset as an example.\n\n# Audience\nThis course is targeted to any life scientist who wants to learn more about the web-based [Galaxy platform](https://usegalaxy.eu/)  and would like to discover its potential to analyze NGS data.\n\n# Learning objectives\nAfter the course, participants should be able to use a public Galaxy server for their research and be able to:\n\n* reproduce and share an analysis in Galaxy\n* run tools such as FastQC, cutadapt, STAR\n* understand their outputs\n* interpret the biological results obtained with Galaxy\n\n# Prerequisites\n##### Knowledge / competencies\nNo command line experience or programming knowledge is required. However, familiarity with NGS and genomic data is highly recommended.\n\n##### Technical\nThe participants need to come with their Wi-Fi enabled laptop with Firefox, Chrome, or Safari web browser installed. There will be access to the eduroam (and guest) networks.\n\n# Application\n\n\nRegistration fees are **60 CHF** for academics and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed. \n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **20/04/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nIf the sanitary situation allows it, this course will be held in person at the University of Lausanne  or at the EPFL. Otherwise it will be live-streamed online.\n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n#  Additional information\nTrainers: Lucille Delisle (EPFL) and Hans-Rudolf Hotz (FMI, SIB)\n\nCoordination: Monique Zahn, SIB Training group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220427_XXX14","@id":"https://www.sib.swiss/training2/website/course/20220427_XXX14","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Galaxy introduction for life scientists"}],"courseMode":["onsite","synchronous"],"keywords":"basic research,experimental biology,interoperability,next generation sequencing,reproducibility,training,michael stadler group","location":"Lausanne","endDate":"2022-04-27","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Lucille Delisle"},{"@type":"Person","name":"Hans-Rudolf Hotz"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-04-27","duration":"PT7H"}]},{"@context":"https://schema.org","@type":"Course","name":"Intermediate Python","description":"\n# Overview\nPython is an open-source and general-purpose scripting language which runs on all major operating systems. It was designed to be easily read and written with comparatively simple syntax. Over the recent years Python has become a programming language of choice for bioinformatics and data analysis, and in particular for applications that make use of machine learning or deep-learning. \nHowever, these applications usually require a good mastering of a few modules (such as numpy, or pandas) that can go beyond basic Python commands.\nThis 1-day course will introduce modules and recipes to unlock the potential of Python for day-to-day data exploration and analysis of real-life data-sets.\n\nTopics that will be covered in this course include:\n* Parsing, transforming, and exploring data using Pandas\n* Performing statistical simulation and testing with Numpy/Scipy\n* Representing data in an efficient and impactful manner using Seaborn\n* Speeding-up your Python code with Numba and more\n\n\n\n# Audience\nThis course is addressed to life scientists, bioinformaticians and researchers who are familiar with writing Python code and core Python elements, and would like to explore further data wrangling and exploration tasks. \n\n# Learning outcomes\nAt the end of this course, the participants are expected to:\n* Parse any tabulated data set in a couple of lines\n* Summarize and perform quality control on their data\n* Filter, sub-sample or aggregate specific parts of their data-set\n* Generate clear visual representations to explore data and communicate their findings\n* Implement best-practice as well as recipes to speed-up key bottlenecks in Python programs\n\n\n# Prerequisites\n#### Knowledge / competencies\nThe course is targeted to life scientists, bioinformaticians, and researchers who are already familiar with the Python programming language and who have basic knowledge on statistics. The competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20220504_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20220620_STATP) and [Introduction to statistics with R](https://www.sib.swiss/training/course/20220207_STATR).\nTest your skills with Python and statistics with [the quiz here](https://docs.google.com/forms/d/e/1FAIpQLSc3vvhtHoY0AeFk1ZJ3CBgQdRtCRf3JNNMfWxMaxYVsfljpcA/viewform), before applying. We recommend 4 out of 6 correct answers.\n\n\n#### Technical\nYou are required to use your own laptop, with a recent Python 3 version. \nThe following modules should be installed on your computer (using [conda](https://conda.io/projects/conda/en/latest/user-guide/concepts/installing-with-conda.html) for example):\n* Jupyter Notebook\n* Pandas\n* Seaborn\n* Numba\n\n\n# Application\nThe registration fees for academics are **60 CHF** and **300 CHF** for for-profit companies.While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **18/05/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will be held in Bern.\n\nThe course will start at 9:00 and end around 17:00. \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 1-day course will teach to write Python code to automate data parsing, perform basic statistical operations and graphical representations.","educationalLevel":"Intermediate","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Intermediate Python","description":"\n# Overview\nPython is an open-source and general-purpose scripting language which runs on all major operating systems. It was designed to be easily read and written with comparatively simple syntax. Over the recent years Python has become a programming language of choice for bioinformatics and data analysis, and in particular for applications that make use of machine learning or deep-learning. \nHowever, these applications usually require a good mastering of a few modules (such as numpy, or pandas) that can go beyond basic Python commands.\nThis 1-day course will introduce modules and recipes to unlock the potential of Python for day-to-day data exploration and analysis of real-life data-sets.\n\nTopics that will be covered in this course include:\n* Parsing, transforming, and exploring data using Pandas\n* Performing statistical simulation and testing with Numpy/Scipy\n* Representing data in an efficient and impactful manner using Seaborn\n* Speeding-up your Python code with Numba and more\n\n\n\n# Audience\nThis course is addressed to life scientists, bioinformaticians and researchers who are familiar with writing Python code and core Python elements, and would like to explore further data wrangling and exploration tasks. \n\n# Learning outcomes\nAt the end of this course, the participants are expected to:\n* Parse any tabulated data set in a couple of lines\n* Summarize and perform quality control on their data\n* Filter, sub-sample or aggregate specific parts of their data-set\n* Generate clear visual representations to explore data and communicate their findings\n* Implement best-practice as well as recipes to speed-up key bottlenecks in Python programs\n\n\n# Prerequisites\n#### Knowledge / competencies\nThe course is targeted to life scientists, bioinformaticians, and researchers who are already familiar with the Python programming language and who have basic knowledge on statistics. The competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20220504_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20220620_STATP) and [Introduction to statistics with R](https://www.sib.swiss/training/course/20220207_STATR).\nTest your skills with Python and statistics with [the quiz here](https://docs.google.com/forms/d/e/1FAIpQLSc3vvhtHoY0AeFk1ZJ3CBgQdRtCRf3JNNMfWxMaxYVsfljpcA/viewform), before applying. We recommend 4 out of 6 correct answers.\n\n\n#### Technical\nYou are required to use your own laptop, with a recent Python 3 version. \nThe following modules should be installed on your computer (using [conda](https://conda.io/projects/conda/en/latest/user-guide/concepts/installing-with-conda.html) for example):\n* Jupyter Notebook\n* Pandas\n* Seaborn\n* Numba\n\n\n# Application\nThe registration fees for academics are **60 CHF** and **300 CHF** for for-profit companies.While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **18/05/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\nThis course will be held in Bern.\n\nThe course will start at 9:00 and end around 17:00. \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220603_XXX16","@id":"https://www.sib.swiss/training2/website/course/20220603_XXX16","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Intermediate Python"}],"courseMode":["onsite","synchronous"],"keywords":"programming,training,data visualisation,torsten schwede & thierry sengstag group,mark ibberson group","location":"Bern","endDate":"2022-06-03","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"},{"@type":"Person","name":"Robin Engler"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-06-03","duration":"PT7H"},{"@type":"CourseInstance","name":"Intermediate Python","description":"**This course is now full with a waiting list.**\n# Overview\nPython is an open-source and general-purpose scripting language which runs on all major operating systems. It was designed to be easily read and written with comparatively simple syntax. Over the recent years Python has become a programming language of choice for bioinformatics and data analysis, and in particular for applications that make use of machine learning or deep-learning. \nHowever, these applications usually require a good mastering of a few modules (such as numpy, or pandas) that can go beyond basic Python commands.\nThis 1-day course will introduce modules and recipes to unlock the potential of Python for day-to-day data exploration and analysis of real-life data-sets.\n\nTopics that will be covered in this course include:\n* Parsing, transforming, and exploring data using pandas\n* Performing statistical simulation and testing with numpy/scipy\n* Representing data in an efficient and impactful manner using seaborn\n* Speeding-up your Python code with numba and more\n\n\n\n# Audience\nThis course is addressed to life scientists, bioinformaticians and researchers who are familiar with writing Python code and core Python elements, and would like to explore if further in their daily data wrangling and exploration tasks. \n\n# Learning outcomes\nAt the end of this course, the participants are expected to:\n* Parse any tabulated data set in a couple of lines \n* Summarize and perform quality control on their data \n* Filter, sub-sample or aggregate specific parts of their data-set \n* Generate clear visual representations to explore data and communicate their findings \n\n\n# Prerequisites\n#### Knowledge / competencies\nThe course is targeted to life scientists, bioinformaticians, and researchers who are already familiar with the Python programming language and who have basic knowledge on statistics. The competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20220928_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20220620_STATP) and [Introduction to statistics with R](https://www.sib.swiss/training/course/20220207_STATR).\nTest your skills with Python and statistics with [the quiz here](https://forms.gle/iCydNS8LUUkm7csz7), before registering. We recommend 4 out of 6 correct answers.\n\nA few days before the course, the students will receive a small \"warm-up\" jupyter notebook to go through. This will be in order to help them get a quick refresher on their python know-how and check that all libraries are working properly. \n\n\n#### Technical\nYou are required to use your own laptop, with a recent Python 3 version. \nThe following modules should be installed on your computer (using [conda](https://conda.io/projects/conda/en/latest/user-guide/concepts/installing-with-conda.html) for example):\n* Jupyter Notebook\n* Sandas\n* Seaborn\n* Numba\n\n\n# Application\nThe registration fees for academics are **60 CHF** and **300 CHF** for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **06/10/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed using Zoom.\n\nThe course will start at 9:00 and end around 17:00. \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Valeria Di Cola,  SIB Training Group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221020_XXX16","@id":"https://www.sib.swiss/training2/website/course/20221020_XXX16","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Intermediate Python"}],"courseMode":["onsite","synchronous"],"keywords":"programming,training,data visualisation,torsten schwede & thierry sengstag group,mark ibberson group","location":"Streamed","endDate":"2022-10-20","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"},{"@type":"Person","name":"Robin Engler"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":120,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-20","duration":"PT7H"}]},{"@context":"https://schema.org","@type":"Course","name":"Introduction to Machine Learning with Python","description":"\n\n\n\n\n\n# Overview\nWith the rise of new technologies, the volume of omics data in the fields of biology and medicine has grown exponentially in recent times and a major issue is to mine useful predictive knowledge from these data. Machine learning (ML) is a discipline in which computer algorithms perform automated learning by using data in order to assist humans to deal with the large volume of multidimensional data. The analysis of such  data is not trivial and ML is a necessary tool to extract knowledge and make predictions that can advance the field of bioinformatics.\n\nThis 2-day course will introduce participants to common ML algorithms and teach how to apply them to omics data in extensive practical sessions. The practical sessions will be conducted in Python3 based on the widely applied scikit-learn ML framework. The course will comprise a number of hands-on exercises and challenges where the participants will acquire a first understanding of the standard ML methods and processes, as well as the practical skills in applying them to real world problems using publicly available biological or medical  data sets.\n\n# Audience\nThis course is intended for PhD students, post-docs and staff scientists who are interested in applying ML to analyze these data.\n\n# Learning objectives\nAt the end of the course, the participants are expected to:\n* Understand the ML taxonomy and the commonly used machine learning algorithms for analysing “omics” data\n* Understand differences between ML approaches and in which situations they can be applied\n* Understand and critically evaluate applications of ML in omics studies\n* Learn how to implement common ML algorithms using the scikit-learn Python framework \n* Interpret and visualize the results obtained from ML analyses\n\n# Prerequisites\n##### Knowledge / competencies\nNo prior knowledge of ML concepts and methods is required. \n\nKnowledge of different -omics data is recommended.\n\nFamiliarity with the Python programming language as well as a basic knowledge on statistics is required. \n\nThe competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20210929_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20210621_STATP) and [Introduction to statistics with R](https://www.sib.swiss/training/course/2021-02-intro-stats). \nTest your skills with Python and statistics with the quiz [here](https://forms.gle/ZpQFyHHwoPQKJSwv7), before registering.\n\n##### Technical\nYou are required to have your own computer with an Internet connection and the following tools installed PRIOR to the course:\n\n*\tlatest Python 3 distribution, preferably bundled using [conda](https://docs.continuum.io/anaconda/install/) \n*\t[Jupyter](https://jupyter.org/install)\n*\tthe [scipy](https://www.scipy.org/install.html) library (NB: if you installed conda, then this library is already installed)\n*\t[scikitLearn](https://scikit-learn.org/stable/install.html) \n\nThere will be access to the eduroam and guest-unil networks.\n\n\n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for registration and free-of-charge cancellation is set is set to **09/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThe course will take place **in person only** at the University of Lausanne  (Metro M1 line, Sorge station). It will NOT be streamed simulataneously. It will start at 9:00 CET and end around 17:00 CET. \n\nPrecise information will be provided to the participants in due time.\n\n#  Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nWe will recommend 0.50 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 2-day course will introduce participants to common ML algorithms and teach how to apply them to omics data in extensive practical sessions.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Introduction to Machine Learning with Python","description":"\n\n\n\n\n\n# Overview\nWith the rise of new technologies, the volume of omics data in the fields of biology and medicine has grown exponentially in recent times and a major issue is to mine useful predictive knowledge from these data. Machine learning (ML) is a discipline in which computer algorithms perform automated learning by using data in order to assist humans to deal with the large volume of multidimensional data. The analysis of such  data is not trivial and ML is a necessary tool to extract knowledge and make predictions that can advance the field of bioinformatics.\n\nThis 2-day course will introduce participants to common ML algorithms and teach how to apply them to omics data in extensive practical sessions. The practical sessions will be conducted in Python3 based on the widely applied scikit-learn ML framework. The course will comprise a number of hands-on exercises and challenges where the participants will acquire a first understanding of the standard ML methods and processes, as well as the practical skills in applying them to real world problems using publicly available biological or medical  data sets.\n\n# Audience\nThis course is intended for PhD students, post-docs and staff scientists who are interested in applying ML to analyze these data.\n\n# Learning objectives\nAt the end of the course, the participants are expected to:\n* Understand the ML taxonomy and the commonly used machine learning algorithms for analysing “omics” data\n* Understand differences between ML approaches and in which situations they can be applied\n* Understand and critically evaluate applications of ML in omics studies\n* Learn how to implement common ML algorithms using the scikit-learn Python framework \n* Interpret and visualize the results obtained from ML analyses\n\n# Prerequisites\n##### Knowledge / competencies\nNo prior knowledge of ML concepts and methods is required. \n\nKnowledge of different -omics data is recommended.\n\nFamiliarity with the Python programming language as well as a basic knowledge on statistics is required. \n\nThe competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20210929_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20210621_STATP) and [Introduction to statistics with R](https://www.sib.swiss/training/course/2021-02-intro-stats). \nTest your skills with Python and statistics with the quiz [here](https://forms.gle/ZpQFyHHwoPQKJSwv7), before registering.\n\n##### Technical\nYou are required to have your own computer with an Internet connection and the following tools installed PRIOR to the course:\n\n*\tlatest Python 3 distribution, preferably bundled using [conda](https://docs.continuum.io/anaconda/install/) \n*\t[Jupyter](https://jupyter.org/install)\n*\tthe [scipy](https://www.scipy.org/install.html) library (NB: if you installed conda, then this library is already installed)\n*\t[scikitLearn](https://scikit-learn.org/stable/install.html) \n\nThere will be access to the eduroam and guest-unil networks.\n\n\n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for registration and free-of-charge cancellation is set is set to **09/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThe course will take place **in person only** at the University of Lausanne  (Metro M1 line, Sorge station). It will NOT be streamed simulataneously. It will start at 9:00 CET and end around 17:00 CET. \n\nPrecise information will be provided to the participants in due time.\n\n#  Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nWe will recommend 0.50 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220616_XXX20","@id":"https://www.sib.swiss/training2/website/course/20220616_XXX20","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Introduction to Machine Learning with Python"}],"courseMode":["onsite","synchronous"],"keywords":"data mining,machine learning,training,mark ibberson group,torsten schwede & thierry sengstag group","location":"Lausanne","endDate":"2022-06-17","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"},{"@type":"Person","name":"Markus Mueller"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-06-16","duration":"P1DT6H"},{"@type":"CourseInstance","name":"Introduction to Machine Learning with Python","description":"**This course is now full with a waiting list.**\n\n\n# Overview\nWith the rise of new technologies, the volume of omics data in the fields of biology and medicine has grown exponentially in recent times and a major issue is to mine useful predictive knowledge from these data. Machine learning (ML) is a discipline in which computer algorithms perform automated learning by using data in order to assist humans to deal with the large volume of multidimensional data. The analysis of such data is not trivial and ML is a necessary tool to extract knowledge and make predictions that can advance the field of bioinformatics. \n\nThis 2-day course will introduce participants to common ML algorithms and teach how to apply them to omics data in extensive practical sessions. The practical sessions will be conducted in Python3 based on the widely applied scikit-learn ML framework. The course will comprise a number of hands-on exercises and challenges where the participants will acquire a first understanding of the standard ML methods and processes, as well as the practical skills in applying them to real world problems using publicly available biological or medical data sets. \n\n# Audience\nThis course is intended for PhD students, post-docs and staff scientists who are interested in applying ML to analyze these data.\n\n# Learning objectives\nAt the end of the course, the participants are expected to:\n* Understand the ML taxonomy and the commonly used machine learning algorithms for analysing “omics” data\n* Understand differences between ML approaches and in which situations they can be applied\n* Understand and critically evaluate applications of ML in omics studies\n* Learn how to implement common ML algorithms using the scikit-learn Python framework \n* Interpret and visualize the results obtained from ML analyses\n\n# Prerequisites\n##### Knowledge / competencies\nNo prior knowledge of ML concepts and methods is required. \n\nKnowledge of different -omics data is recommended.\n\nFamiliarity with the Python programming language and pandas dataframes as well as a basic knowledge on statistics is required. \n\nThe competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20220504_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20220620_STATP) and [Introduction to statistics with R](https://www.sib.swiss/training/course/20220207_STATR). \nTest your skills with Python and statistics with the quiz [here](https://forms.gle/ZpQFyHHwoPQKJSwv7), before registering.\n\n##### Technical\nThis course will be streamed, you are thus required to have your own computer with an Internet connection.\n\nAdditionally, you will need to have a recent python3 as well as a number of python libraries installed. Please follow these [instructions to setup your environment ](https://github.com/sib-swiss/intro-machine-learning-training/blob/main/env_setup.md)(note: these instructions use [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) to manage the different packages) \n\nPlease perform these installations PRIOR to the course and contact us if you have any trouble. \n\nFinally, although not mandatory, we also highly recommend you to use the same computer to connect to the zoom classroom and perform the exercises, otherwise we will have difficulties helping you debug your code. \n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for registration and free-of-charge cancellation is set is set to **19/09/2022**. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed using Zoom. \n\nThe course will start at 9:00 CET and end around 17:00 CET. \n\nPrecise information will be provided to the participants in due time.\n\n#  Additional information\nCoordination: Valeria Di Cola, SIB Training Group.\n\nWe will recommend 0.50 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto:training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221003_XXX20","@id":"https://www.sib.swiss/training2/website/course/20221003_XXX20","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Introduction to Machine Learning with Python"}],"courseMode":["onsite","synchronous"],"keywords":"data mining,machine learning,training,mark ibberson group,mauro delorenzi & frédéric schütz group","location":"Streamed","endDate":"2022-10-04","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin and Markus Mueller"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-03","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"Introduction to RNA-Seq: From quality control to pathway analysis","description":"\n\nThis course is now full with a waiting list. We are no longer accepting applications. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our [courses mailing list](https://lists.sib.swiss/mailman/listinfo/courses), if not yet done. Thank you for your understanding.\n\n# Overview\nThis two-day course will present the theory and bioinformatics tools required to analyze RNA-seq gene expression data, The course will present the theory and bioinformatics tools required to analyse RNA-seq gene expression data, from the raw data until the biological interpretation. The following topics will be discussed:\n* Quality control\n* Mapping to genomes & transcriptomes\n* Read counting, gene & exons differential expression analysis\n* GO enrichment and pathway analysis\n\n# Audience\nThis course is addressed to life scientists or bioinformaticians familiar with \"Next Generation Sequencing\" who wish to acquire the necessary skills to analyse RNA-seq gene expression data.\n\n# Learning outcomes\nAt the end of the course, the participants are expected to:\n* Describe advantages and pitfalls of RNA sequencing\n* Design their own experiment\n* Perform the downstream analysis using command line software (QC, mapping, counting, differential expression analysis, pathway analysis, etc)\n* Critically assess the quality of their results at each step of the downstream analysis\n\n# Prerequisites\n##### Knowledge / competencies\nParticipants should already have a basic knowledge in Next Generation Sequencing (NGS) techniques.\n\nBasic knowledge of the R statistical software and completion of an introductory course on UNIX (eg UNIX fundamentals) or equivalent knowledge is also required.\n##### Technical\nYou are required to have your own Wi-Fi enabled laptop.\n\nYou should also have access to a terminal on your computer: \n\n macOS / Linux: you can use the default terminal. If you are unfamiliar with scp, you may want to install a graphical client for file transfer, such as fileZilla \nWindows, you should install a terminal which lets you do ssh (for instance mobaXterm).\n\n\n# Schedule\n\nThis course will be held ONLY in person at the University of Basel.\n\nThe course will start at 9:00 and end around 17:00.\n\nPrecise information will be provided to the participants in due time.\n\n# Application\n\n\nThis course is now full with a waiting list. We are not accepting applications any more.\n\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies.  \nWhile participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **14/08/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n#  Additional information\nCoordination: Diana Marek, SIB Training Group.\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 2-day course presents theory and bioinformatics tools required to analyze RNA-seq gene expression data, from the raw data to the biological interpreta","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Introduction to RNA-Seq: From quality control to pathway analysis","description":"\n\nThis course is now full with a waiting list. We are no longer accepting applications. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of [upcoming events](https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our [courses mailing list](https://lists.sib.swiss/mailman/listinfo/courses), if not yet done. Thank you for your understanding.\n\n# Overview\nThis two-day course will present the theory and bioinformatics tools required to analyze RNA-seq gene expression data, The course will present the theory and bioinformatics tools required to analyse RNA-seq gene expression data, from the raw data until the biological interpretation. The following topics will be discussed:\n* Quality control\n* Mapping to genomes & transcriptomes\n* Read counting, gene & exons differential expression analysis\n* GO enrichment and pathway analysis\n\n# Audience\nThis course is addressed to life scientists or bioinformaticians familiar with \"Next Generation Sequencing\" who wish to acquire the necessary skills to analyse RNA-seq gene expression data.\n\n# Learning outcomes\nAt the end of the course, the participants are expected to:\n* Describe advantages and pitfalls of RNA sequencing\n* Design their own experiment\n* Perform the downstream analysis using command line software (QC, mapping, counting, differential expression analysis, pathway analysis, etc)\n* Critically assess the quality of their results at each step of the downstream analysis\n\n# Prerequisites\n##### Knowledge / competencies\nParticipants should already have a basic knowledge in Next Generation Sequencing (NGS) techniques.\n\nBasic knowledge of the R statistical software and completion of an introductory course on UNIX (eg UNIX fundamentals) or equivalent knowledge is also required.\n##### Technical\nYou are required to have your own Wi-Fi enabled laptop.\n\nYou should also have access to a terminal on your computer: \n\n macOS / Linux: you can use the default terminal. If you are unfamiliar with scp, you may want to install a graphical client for file transfer, such as fileZilla \nWindows, you should install a terminal which lets you do ssh (for instance mobaXterm).\n\n\n# Schedule\n\nThis course will be held ONLY in person at the University of Basel.\n\nThe course will start at 9:00 and end around 17:00.\n\nPrecise information will be provided to the participants in due time.\n\n# Application\n\n\nThis course is now full with a waiting list. We are not accepting applications any more.\n\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies.  \nWhile participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for free-of-charge cancellation is set to **14/08/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n#  Additional information\nCoordination: Diana Marek, SIB Training Group.\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220901_XXX23","@id":"https://www.sib.swiss/training2/website/course/20220901_XXX23","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Introduction to RNA-Seq: From quality control to pathway analysis"}],"courseMode":["onsite","synchronous"],"keywords":"functional genomics,genes and genomes,next generation sequencing,transcriptomics,training,torsten schwede & thierry sengstag group","location":"Basel","endDate":"2022-09-02","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-09-01","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"Introduction to statistics with Python","description":"# Overview\nStatistics are an integral aspect of scientific research, particularly for the life sciences which rely heavily on quantitative methodologies. \nThis course is designed to provide researchers in the life sciences with an introduction to statistics and statistical concepts and their application to a variety of biological problems.\n\nThe course will combine lectures on statistics, short tutorials and practical exercises on the topics discussed in the class. These practical exercises will be implemented in the widely used Python language and environment for statistical computing and graphics.\n\nTopics covered during the course include: \n* Data manipulation, summarization and plotting : reminder/intro to data mamipulation in Python\n* Hypothesis testing : when to use it and how?\n* Correlation and regression analysis\n\n# Audience\nThis course is intended for life scientists from all levels and disciplines who are not experts in statistics. \n\nAlthough we will provide materials and a reminder on data mamipulation in Python, participants must be comfortable with the Python environment and be able to read, understand and write basic Python commands before attending this course.\n\n# Learning objectives\nAt the end of the course, participants should be able to :\n* explore their data, choose the right method to summarize it, graphically and numerically.\n* perform appropriate basic hypothesis tests on a datatest.\n* appraise the results and limitations of these tests, their underlying hypotheses, and multiple testing issues.\n* assess and describe the link between different variables, using correlation and regression analysis.\n\n# Prerequisites\n##### Knowledge / competencies\n* No prior statistical knowledge is required in order to attend the course\n* Completion of a basic/introductory course on Python \n\nBefore registering, you may [test your Python skills](https://forms.gle/oZxMeopsmpxst76g9) this quizz. \nWe recommend a score of at least 3/5 in order to benefit from the course.\n\n##### Technical\n\nPlease make sure that the following software is installed on your machine  PRIOR to the course: \n* latest python3 distribution, preferably bundled using [conda](https://docs.continuum.io/anaconda/install/)\n* [jupyter](https://jupyter.org/install)\n* the [scipy](https://www.scipy.org/install.html) library (NB: if you installed conda, then this library is already installed)\n\n\n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **09/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will take place at the University of Basel. More details will be communicated to the participants.\n\n\nThe course will start at 9:00 and end around 17:00. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Grégoire Rossier\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":null,"educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Introduction to statistics with Python","description":"# Overview\nStatistics are an integral aspect of scientific research, particularly for the life sciences which rely heavily on quantitative methodologies. \nThis course is designed to provide researchers in the life sciences with an introduction to statistics and statistical concepts and their application to a variety of biological problems.\n\nThe course will combine lectures on statistics, short tutorials and practical exercises on the topics discussed in the class. These practical exercises will be implemented in the widely used Python language and environment for statistical computing and graphics.\n\nTopics covered during the course include: \n* Data manipulation, summarization and plotting : reminder/intro to data mamipulation in Python\n* Hypothesis testing : when to use it and how?\n* Correlation and regression analysis\n\n# Audience\nThis course is intended for life scientists from all levels and disciplines who are not experts in statistics. \n\nAlthough we will provide materials and a reminder on data mamipulation in Python, participants must be comfortable with the Python environment and be able to read, understand and write basic Python commands before attending this course.\n\n# Learning objectives\nAt the end of the course, participants should be able to :\n* explore their data, choose the right method to summarize it, graphically and numerically.\n* perform appropriate basic hypothesis tests on a datatest.\n* appraise the results and limitations of these tests, their underlying hypotheses, and multiple testing issues.\n* assess and describe the link between different variables, using correlation and regression analysis.\n\n# Prerequisites\n##### Knowledge / competencies\n* No prior statistical knowledge is required in order to attend the course\n* Completion of a basic/introductory course on Python \n\nBefore registering, you may [test your Python skills](https://forms.gle/oZxMeopsmpxst76g9) this quizz. \nWe recommend a score of at least 3/5 in order to benefit from the course.\n\n##### Technical\n\nPlease make sure that the following software is installed on your machine  PRIOR to the course: \n* latest python3 distribution, preferably bundled using [conda](https://docs.continuum.io/anaconda/install/)\n* [jupyter](https://jupyter.org/install)\n* the [scipy](https://www.scipy.org/install.html) library (NB: if you installed conda, then this library is already installed)\n\n\n\n# Application\nThe registration fees for academics are **120 CHF** and **600 CHF** for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **09/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will take place at the University of Basel. More details will be communicated to the participants.\n\n\nThe course will start at 9:00 and end around 17:00. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Grégoire Rossier\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220620_XXX24","@id":"https://www.sib.swiss/training2/website/course/20220620_XXX24","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Introduction to statistics with Python"}],"courseMode":["onsite","synchronous"],"keywords":"programming,biostatistics","location":"Basel","endDate":"2022-06-21","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Sébastien Boyer"},{"@type":"Person","name":"Wandrille Duchemin"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-06-20","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"NGS - Genome variant analysis","description":"\n\n\n\n\nOverview\nThe detection of genetic variation is of major interest in various disciplines spanning from ecology and evolution research to inherited disease discovery and precision oncology. Next generation sequencing (NGS) methods are very powerful for the detection of genomic variants. Thanks to its throughput and cost-efficiency it enables the detection of a large number of variants in a large number of samples. In this two-day course we will cover the steps from read alignment to variant calling and annotation. We will mainly focus on the detection of germline mutations by following the GATK best practices. The training materials for this course are in its dedicated GitHub page.\n\nAudience\nThis course is intended for life scientists who are already familiar with general concepts of NGS technologies and want to expand their knowledge and skills on variant analysis.\nLearning objectives\nAt the end of the course participants should be able to:\n\n\tUnderstand important aspects of NGS and read alignment for variant analysis\n\tPerform a read alignment ready for variant analysis\n\tPerform variant calling according to GATK best practices\n\tPerform a variant annotation\n\nPrerequisites\nKnowledge / competencies:\nParticipants should have knowledge in NGS techniques, quality control and alignment to a reference genome. Participants should have a basic understanding of working with command line tools on Unix-based systems. You can test your skills with Unix with the quiz here. If you do not feel comfortable with UNIX commands, please take our Unix fundamentals e-learning module.\n\nTechnical:\nParticipants should have their own computers. \t\n\nApplication\nThe registration fees are 120 CHF for academics and  600 CHF for companies.   While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.  Applications will close once the places will be filled. \nDeadline for registration and free-of-charge cancellation is set to 01/09/2022. Cancellation after this date will not be reimbursed.  Please note that participation to SIB courses is subject to this and other general conditions, available here.\n\nYou will be informed by email of your registration confirmation.\nLocation\nThis course will take place at the University of Bern.\n\nIt will start at 9:15 and end around 17:15.\n\nPrecise information will be provided to the participants in due time.\nSchedule\n\nThe schedule and course materials are in the dedicated [GitHub page](https://sib-swiss.github.io/NGS-variants-training/latest/course_schedule/). \n\nAdditional information\n\nCoordination: Valeria Di Cola, SIB training group\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the session). \n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"In this course we will cover the NGS steps: read alignment to variant calling and annotation, with main focus on the detection of germline mutations.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"NGS - Genome variant analysis","description":"\n\n\n\n\nOverview\nThe detection of genetic variation is of major interest in various disciplines spanning from ecology and evolution research to inherited disease discovery and precision oncology. Next generation sequencing (NGS) methods are very powerful for the detection of genomic variants. Thanks to its throughput and cost-efficiency it enables the detection of a large number of variants in a large number of samples. In this two-day course we will cover the steps from read alignment to variant calling and annotation. We will mainly focus on the detection of germline mutations by following the GATK best practices. The training materials for this course are in its dedicated GitHub page.\n\nAudience\nThis course is intended for life scientists who are already familiar with general concepts of NGS technologies and want to expand their knowledge and skills on variant analysis.\nLearning objectives\nAt the end of the course participants should be able to:\n\n\tUnderstand important aspects of NGS and read alignment for variant analysis\n\tPerform a read alignment ready for variant analysis\n\tPerform variant calling according to GATK best practices\n\tPerform a variant annotation\n\nPrerequisites\nKnowledge / competencies:\nParticipants should have knowledge in NGS techniques, quality control and alignment to a reference genome. Participants should have a basic understanding of working with command line tools on Unix-based systems. You can test your skills with Unix with the quiz here. If you do not feel comfortable with UNIX commands, please take our Unix fundamentals e-learning module.\n\nTechnical:\nParticipants should have their own computers. \t\n\nApplication\nThe registration fees are 120 CHF for academics and  600 CHF for companies.   While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.  Applications will close once the places will be filled. \nDeadline for registration and free-of-charge cancellation is set to 01/09/2022. Cancellation after this date will not be reimbursed.  Please note that participation to SIB courses is subject to this and other general conditions, available here.\n\nYou will be informed by email of your registration confirmation.\nLocation\nThis course will take place at the University of Bern.\n\nIt will start at 9:15 and end around 17:15.\n\nPrecise information will be provided to the participants in due time.\nSchedule\n\nThe schedule and course materials are in the dedicated [GitHub page](https://sib-swiss.github.io/NGS-variants-training/latest/course_schedule/). \n\nAdditional information\n\nCoordination: Valeria Di Cola, SIB training group\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the session). \n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220926_XXX30","@id":"https://www.sib.swiss/training2/website/course/20220926_XXX30","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"NGS - Genome variant analysis"}],"courseMode":["onsite","synchronous"],"keywords":"medicine and health,next generation sequencing,training,agriculture,environmental science,medicine and health","location":"Bern","endDate":"2022-09-27","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Geert van Geest"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-09-26","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"NGS - Quality control, Alignment, Visualisation","description":"\n\n\n\n\n# Overview\nNext Generation Sequencing (NGS) methods have had a large impact on many biological fields due to a very rapid decrease in cost and throughput. It often results in hundreds of gigabytes of data. This can make the downstream analysis challenging and requires bioinformatics skills.\n\nIn this module, we will introduce the most used sequencing technologies and explain their concepts. Using different datasets, we will practice quality control, alignment of reads to a reference genome and visualize the output.\n\nThe course materials can be found in the dedicated [GitHub](https://sib-swiss.github.io/NGS-introduction-training/latest/) page.\n\n# Audience\nThis course is intended for life scientists who are already dealing with NGS data and would like to be able to start analysing them.\n\n# Learning Objectives\nAt the end of the course participants should be able to:\n* understand the basics of the different NGS technologies\n* perform quality control for better downstream analysis \n* align reads to a reference genome \n* visualize the output.\n\n# Prerequisites\n##### Knowledge / competencies:\nParticipants should have a basic understanding of working with the command line on UNIX-based systems like Linux or Mac. If you do not feel comfortable with UNIX commands, please take our [UNIX fundamentals](https://edu.sib.swiss/course/view.php?name=UNIX_I) e-learning module or attend one of the \"[First Steps with UNIX in Life Sciences](https://www.sib.swiss/training/course/20220111_FSWU)\" that we propose in 2022.\n##### Technical:\nParticipants should bring their own computers. Software requirements will be communicated before the course.\n\n# Application\nThe registration fees for academics are **180 CHF** and **900 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close once the places are filled. Deadline for registration and free-of-charge cancellation is set to *02/05/2022*. Cancellation after this date will not be reimbursed.\n\nPlease note that participation to SIB courses is subject to this and other general conditions, available [here](https://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n# Location & Time\nUniversität Bern, Room 028, Hochschulstrasse 4.\n\nThis course will start at 9:15 and end around 17:15. Precise information will be provided to the participants on due time.\n\n\n# Additional information\n\nThe course will be taught by Geert van Geest.\n\n\nCoordination: Valeria Di Cola, SIB Training group.\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"Introduction to the most used sequencing technologies and explain their concepts","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"NGS - Quality control, Alignment, Visualisation","description":"\n\n\n\n\n# Overview\nNext Generation Sequencing (NGS) methods have had a large impact on many biological fields due to a very rapid decrease in cost and throughput. It often results in hundreds of gigabytes of data. This can make the downstream analysis challenging and requires bioinformatics skills.\n\nIn this module, we will introduce the most used sequencing technologies and explain their concepts. Using different datasets, we will practice quality control, alignment of reads to a reference genome and visualize the output.\n\nThe course materials can be found in the dedicated [GitHub](https://sib-swiss.github.io/NGS-introduction-training/latest/) page.\n\n# Audience\nThis course is intended for life scientists who are already dealing with NGS data and would like to be able to start analysing them.\n\n# Learning Objectives\nAt the end of the course participants should be able to:\n* understand the basics of the different NGS technologies\n* perform quality control for better downstream analysis \n* align reads to a reference genome \n* visualize the output.\n\n# Prerequisites\n##### Knowledge / competencies:\nParticipants should have a basic understanding of working with the command line on UNIX-based systems like Linux or Mac. If you do not feel comfortable with UNIX commands, please take our [UNIX fundamentals](https://edu.sib.swiss/course/view.php?name=UNIX_I) e-learning module or attend one of the \"[First Steps with UNIX in Life Sciences](https://www.sib.swiss/training/course/20220111_FSWU)\" that we propose in 2022.\n##### Technical:\nParticipants should bring their own computers. Software requirements will be communicated before the course.\n\n# Application\nThe registration fees for academics are **180 CHF** and **900 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close once the places are filled. Deadline for registration and free-of-charge cancellation is set to *02/05/2022*. Cancellation after this date will not be reimbursed.\n\nPlease note that participation to SIB courses is subject to this and other general conditions, available [here](https://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n# Location & Time\nUniversität Bern, Room 028, Hochschulstrasse 4.\n\nThis course will start at 9:15 and end around 17:15. Precise information will be provided to the participants on due time.\n\n\n# Additional information\n\nThe course will be taught by Geert van Geest.\n\n\nCoordination: Valeria Di Cola, SIB Training group.\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220516_XXX31","@id":"https://www.sib.swiss/training2/website/course/20220516_XXX31","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"NGS - Quality control, Alignment, Visualisation"}],"courseMode":["onsite","synchronous"],"keywords":"training,genes and genomes,next generation sequencing,environmental science,medicine and health","location":"Bern","endDate":"2022-05-18","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Geert van Geest"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":180,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":900,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-05-16","duration":"P2DT5H"},{"@type":"CourseInstance","name":"NGS - Quality control, Alignment, Visualisation","description":"\n\n\n\n\n# Overview\nNext Generation Sequencing (NGS) methods have had a large impact on many biological fields due to a very rapid decrease in cost and throughput. It often results in hundreds of gigabytes of data. This can make the downstream analysis challenging and requires bioinformatics skills.\n\nIn this module, we will introduce the most used sequencing technologies and explain their concepts. Using different datasets, we will practice quality control, alignment of reads to a reference genome and visualize the output.\n\nThe course materials can be found in the dedicated [GitHub](https://sib-swiss.github.io/NGS-introduction-training/latest/) page.\n\n# Audience\nThis course is intended for life scientists who are already dealing with NGS data and would like to be able to start analysing them.\n\n# Learning Objectives\nAt the end of the course participants should be able to:\n* understand the basics of the different NGS technologies\n* perform quality control for better downstream analysis \n* align reads to a reference genome \n* visualize the output.\n\n# Prerequisites\n##### Knowledge / competencies:\nParticipants should have a basic understanding of working with the command line on UNIX-based systems like Linux or Mac. If you do not feel comfortable with UNIX commands, please take our [UNIX fundamentals](https://edu.sib.swiss/course/view.php?name=UNIX_I) e-learning module or attend one of the \"[First Steps with UNIX in Life Sciences](https://www.sib.swiss/training/course/20220111_FSWU)\" that we propose in 2022.\n##### Technical:\nParticipants should bring their own computers. Software requirements will be communicated before the course.\n\n# Application\nThe registration fees for academics are **180 CHF** and **900 CHF** for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for registration and free-of-charge cancellation is set to *07/11/2022*. Cancellation after this date will not be reimbursed.\n\nPlease note that participation to SIB courses is subject to this and other general conditions, available [here](https://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n# Location & Time\nThis course will be streamed using Zoom.\nThis course will start at 9:00 and end around 17:00. Precise information will be provided to the participants on due time.\n\n\n# Additional information\n\nThe course will be taught by Geert van Geest.\n\n\nCoordination: Valeria Di Cola, SIB Training group.\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221121_XXX31","@id":"https://www.sib.swiss/training2/website/course/20221121_XXX31","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"NGS - Quality control, Alignment, Visualisation"}],"courseMode":["onsite","synchronous"],"keywords":"training,genes and genomes,next generation sequencing,environmental science,medicine and health","location":"Streamed","endDate":"2022-11-23","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Geert van Geest"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":180,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":900,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-11-21","duration":"P2DT5H"}]},{"@context":"https://schema.org","@type":"Course","name":"Protein sequence databases and sequence annotation at UniProtKB","description":"**This course will be streamed only for registered participants. Registered participants will receive information from the course organizer in due time.**\n\n# Overview\nUsing high-throughput technologies, you can identify long lists of candidate genes that differ between two experimental conditions. In order to interpret these gene lists and to discover fundamental properties like gene function and disease relevance, you need to use the annotation linked to a given gene or protein sequence.\n\nThe goals of this course are to give some basic theoretical and practical knowledge on protein sequence databases with a focus on [UniProtKB](https://www.uniprot.org/), on the different manual and automated annotation pipelines (such as [HAMAP](https://hamap.expasy.org/)) and, in particular, on the optimum use of UniProtKB. UniProtKB and HAMAP are SIB resources; they are listed in [Expasy](https://www.expasy.org/), the Swiss Bioinformatics Portal. During the lecture and exercise  sessions, we will cover questions such as:\n* Where do the protein sequences come from?\n* What are the differences between the major protein sequence databases?\n* What are the manual and automated gene / protein annotation pipelines?\n* What are the Gene Ontology (GO) annotation pipelines?\n* How to assess protein sequence accuracy and annotation quality?\n* How to extract biological knowledge from a Blast result or gene list?\n\n# Audience\nThis course targets biologists and bioinformaticians who seek to analyze protein data. It will also be useful to programmers and data scientists, be they from academia or industry, who programmatically access protein sequence databases and need to understand the data.\n\n# Learning outcomes\nAt the end of the course, the participants are expected to:\n* list the differences between the major protein sequence databases\n* describe the major protein sequence and GO annotation pipelines\n* assess the accuracy of a protein sequence and the quality of annotation\n* use the query interfaces on the UniProtKB website to make meaningful and productive requests \n\n# Prerequisites\n##### Knowledge / competencies\nThis course is designed for beginners. There are no requirements.\n##### Technical\nThis course will be streamed, you are thus required to have your own computer with an Internet connection.\n\n# Application\nThe registration fees for academics are **60 CHF** and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed. \n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **07/10/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed using Zoom. \n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Monique Zahn, SIB training group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This 1-day course presents theoretical and practical knowledge on protein sequence databases and annotation pipelines, with a focus on UniProtKB.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Protein sequence databases and sequence annotation at UniProtKB","description":"**This course will be streamed only for registered participants. Registered participants will receive information from the course organizer in due time.**\n\n# Overview\nUsing high-throughput technologies, you can identify long lists of candidate genes that differ between two experimental conditions. In order to interpret these gene lists and to discover fundamental properties like gene function and disease relevance, you need to use the annotation linked to a given gene or protein sequence.\n\nThe goals of this course are to give some basic theoretical and practical knowledge on protein sequence databases with a focus on [UniProtKB](https://www.uniprot.org/), on the different manual and automated annotation pipelines (such as [HAMAP](https://hamap.expasy.org/)) and, in particular, on the optimum use of UniProtKB. UniProtKB and HAMAP are SIB resources; they are listed in [Expasy](https://www.expasy.org/), the Swiss Bioinformatics Portal. During the lecture and exercise  sessions, we will cover questions such as:\n* Where do the protein sequences come from?\n* What are the differences between the major protein sequence databases?\n* What are the manual and automated gene / protein annotation pipelines?\n* What are the Gene Ontology (GO) annotation pipelines?\n* How to assess protein sequence accuracy and annotation quality?\n* How to extract biological knowledge from a Blast result or gene list?\n\n# Audience\nThis course targets biologists and bioinformaticians who seek to analyze protein data. It will also be useful to programmers and data scientists, be they from academia or industry, who programmatically access protein sequence databases and need to understand the data.\n\n# Learning outcomes\nAt the end of the course, the participants are expected to:\n* list the differences between the major protein sequence databases\n* describe the major protein sequence and GO annotation pipelines\n* assess the accuracy of a protein sequence and the quality of annotation\n* use the query interfaces on the UniProtKB website to make meaningful and productive requests \n\n# Prerequisites\n##### Knowledge / competencies\nThis course is designed for beginners. There are no requirements.\n##### Technical\nThis course will be streamed, you are thus required to have your own computer with an Internet connection.\n\n# Application\nThe registration fees for academics are **60 CHF** and **300 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed. \n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nDeadline for free-of-charge cancellation is set to **07/10/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed using Zoom. \n\nThe course will start at 9:00 CET and end around 17:00 CET. Precise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Monique Zahn, SIB training group.\n\nWe will recommend 0.25 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221014_XXX33","@id":"https://www.sib.swiss/training2/website/course/20221014_XXX33","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Protein sequence databases and sequence annotation at UniProtKB"}],"courseMode":["onsite","synchronous"],"keywords":"alan bridge group,database curation,ontology,proteins and proteomes,training","location":"Streamed","endDate":"2022-10-14","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Elisabeth Gasteiger"},{"@type":"Person","name":"Marie-Claude Blatter"},{"@type":"Person","name":"Ivo Pedruzzi"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":60,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":300,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-14","duration":"PT7H"}]},{"@context":"https://schema.org","@type":"Course","name":"Research Data Management and Data Management Plan","description":"\n\n\n\n\n\n\n\n# Overview\nRecent studies have shown that worldwide, between 51% and 89% of published life sciences research is not reproducible, with consequent losses estimated at around $100 billions/year in biomedical research (Chalmers et al., 2009; Freedman et al., 2015; Begley and Ioannidis, 2015). In particular, these studies have made clear that the research data associated with a publication are fundamental to validate the published analyses and results. Many causes contribute to this lack of reproducibility in life science studies such as a lack of rigor in data management and analysis. This extensive problem related to improper research management has urged scientists to consider developing efficient Data Management Plans (DMP) for their research projects, a need that is also reflected in the requirements of funding agencies, amongst which the Swiss National Science Foundation (SNFS) and Horizon 2020. \n\nThis course, given by researchers and professionals involved in Big Data management at SIB/Vital-IT as well as in Data Management Plan preparation at UNIL/CHUV,  will provide you with effective support to produce high quality DMP complying with the guidelines established by funding agencies. Importantly, it will provide you with tools to generate robust data and excellent quality studies that are reproducible and reusable.\n\n*Sources of information*\n* Begley, C G, and Ioannidis, J. PA. “Reproducibility in science improving the standard for basic and preclinical research.” Circulation research. 2015; 116.1: 116-126. DOI: [10.1161/CIRCRESAHA.114.303819](https://doi.org/10.1161/circresaha.114.303819)\n* Chalmers I, Glasziou P. Avoidable Waste in the Production and Reporting of Research Evidence. Lancet. 2009; 374(9683): 86–89.  DOI: [10.1016/S0140-6736(09)60329-9](https://doi.org/10.1016/S0140-6736(09)60329-9)\n* Freedman LP, Cockburn IM, Simcoe TS. The Economics of Reproducibility in Preclinical Research. PLoS Biol. 2015;13(6): e1002165. DOI: [10.1371/journal.pbio.1002165](https://doi.org/10.1371/journal.pbio.1002165)\n\n# Schedule\n**Thursday 23 June 2022- 09:00-17:00 CET**\n\nAt first, participants will be introduced to the notion of research reproducibility and to the need for a Data Management Plan (DMP) preparation, an evolving document reporting how the research data will be managed during and after a research project. You will learn best practices in Research Data Management (RDM) and especially two first steps concerning data collection and data documentation. During the exercises, you will directly apply what you have learned.\n\nDuring the afternoon, you will learn additional steps of the RDM cycle concerning Ethics, Legal, Security issues, Data Preservation and Data Sharing. The remaining part of the course will be dedicated to demos and exercises on Data management, where you will learn how to share your published data on adapted repositories, such as Zenodo, and how to fill a DMP corresponding to your own research using the “SIB/Vital-IT DMP Canvas Generator tool”.\n\n**Friday 24 June 2022 - 09:00-12:30 CET**\n\nOn this half day, you will continue to complete your DPMs and refine the information under the guidance of the teachers. Those who would like will be able to present their draft DMPs in order to concretize their work and generate productive discussions in the group.\n\n# Audience\nThe course is addressed to postgraduate students and researchers who plan to apply for SNFS funds and want to be trained on how to efficiently complete the Data Management Plan form. At the same time, this course aims at educating the participants on Data Management principles in general.\n# Learning objectives\nAt the end of the course you will be able to:\n* put in place a DMP (data management plan) which fulfills the requirements of the funding agencies such as the SNFS and H2020, which require a DMP to be put in place\n* manage in detail your research data, specifying how your data will be analysed, organised, stored, secured and shared\n* specify the type of data that is going to be created and shared\n* indicate the process to be followed in respect of the budget, intellectual property, and monitoring \n* use the [“SIB/Vital-IT DMP Canvas Generator tool”](https://dmp.vital-it.ch) to make your own DMP template. \n\n# Prerequisites\n##### ***Knowledge / competencies***\nTo be involved in Life Sciences research.\n\n##### ***Technical***\nYou will need a laptop with a web browser installed. \n\n\n# Trainers\n* Cécile Lebrand - Open Science advocate and information specialist, Bibliothèque Universitaire de Médecine, CHUV\n\n* Vassilios Ioannidis - SIB/Vital-IT Competence Center, SIB Swiss Institute of Bioinformatics\n\n* Grégoire Rossier - SIB/Vital-IT Competence Center & SIB Training, SIB Swiss Institute of Bioinformatics\n\n# Application\n\n\n\n\nThis course is free of charges. It is partially sponsored by [ELIXIR-CONVERGE](https://elixir-europe.org/about-us/how-funded/eu-projects/converge).\nWhile participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for cancellation is set to **10/06/2022**. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. \n\n # Venue and time\n\n\nThis course will ONLY be held in person at the University of Lausanne (Metro M1 line, Sorge station). **No online streaming will be offered.**\n\nIt will start at 9:00 and end around 12:00 on the second day.\n\nPrecise information will be provided to the participants before the course.\n\n#  Additional information\nCoordination: Diana Marek, SIB Training Group\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).\n\n\n\n\n\n\n\n\nELIXIR-CONVERGE is funded by the European Commission within the Research Infrastructures programme of Horizon 2020\n\n\n\n","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"The course aims at educating researchers  on Data Management principles and train them to submit Data Management Plan form.","educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Research Data Management and Data Management Plan","description":"\n\n\n\n\n\n\n\n# Overview\nRecent studies have shown that worldwide, between 51% and 89% of published life sciences research is not reproducible, with consequent losses estimated at around $100 billions/year in biomedical research (Chalmers et al., 2009; Freedman et al., 2015; Begley and Ioannidis, 2015). In particular, these studies have made clear that the research data associated with a publication are fundamental to validate the published analyses and results. Many causes contribute to this lack of reproducibility in life science studies such as a lack of rigor in data management and analysis. This extensive problem related to improper research management has urged scientists to consider developing efficient Data Management Plans (DMP) for their research projects, a need that is also reflected in the requirements of funding agencies, amongst which the Swiss National Science Foundation (SNFS) and Horizon 2020. \n\nThis course, given by researchers and professionals involved in Big Data management at SIB/Vital-IT as well as in Data Management Plan preparation at UNIL/CHUV,  will provide you with effective support to produce high quality DMP complying with the guidelines established by funding agencies. Importantly, it will provide you with tools to generate robust data and excellent quality studies that are reproducible and reusable.\n\n*Sources of information*\n* Begley, C G, and Ioannidis, J. PA. “Reproducibility in science improving the standard for basic and preclinical research.” Circulation research. 2015; 116.1: 116-126. DOI: [10.1161/CIRCRESAHA.114.303819](https://doi.org/10.1161/circresaha.114.303819)\n* Chalmers I, Glasziou P. Avoidable Waste in the Production and Reporting of Research Evidence. Lancet. 2009; 374(9683): 86–89.  DOI: [10.1016/S0140-6736(09)60329-9](https://doi.org/10.1016/S0140-6736(09)60329-9)\n* Freedman LP, Cockburn IM, Simcoe TS. The Economics of Reproducibility in Preclinical Research. PLoS Biol. 2015;13(6): e1002165. DOI: [10.1371/journal.pbio.1002165](https://doi.org/10.1371/journal.pbio.1002165)\n\n# Schedule\n**Thursday 23 June 2022- 09:00-17:00 CET**\n\nAt first, participants will be introduced to the notion of research reproducibility and to the need for a Data Management Plan (DMP) preparation, an evolving document reporting how the research data will be managed during and after a research project. You will learn best practices in Research Data Management (RDM) and especially two first steps concerning data collection and data documentation. During the exercises, you will directly apply what you have learned.\n\nDuring the afternoon, you will learn additional steps of the RDM cycle concerning Ethics, Legal, Security issues, Data Preservation and Data Sharing. The remaining part of the course will be dedicated to demos and exercises on Data management, where you will learn how to share your published data on adapted repositories, such as Zenodo, and how to fill a DMP corresponding to your own research using the “SIB/Vital-IT DMP Canvas Generator tool”.\n\n**Friday 24 June 2022 - 09:00-12:30 CET**\n\nOn this half day, you will continue to complete your DPMs and refine the information under the guidance of the teachers. Those who would like will be able to present their draft DMPs in order to concretize their work and generate productive discussions in the group.\n\n# Audience\nThe course is addressed to postgraduate students and researchers who plan to apply for SNFS funds and want to be trained on how to efficiently complete the Data Management Plan form. At the same time, this course aims at educating the participants on Data Management principles in general.\n# Learning objectives\nAt the end of the course you will be able to:\n* put in place a DMP (data management plan) which fulfills the requirements of the funding agencies such as the SNFS and H2020, which require a DMP to be put in place\n* manage in detail your research data, specifying how your data will be analysed, organised, stored, secured and shared\n* specify the type of data that is going to be created and shared\n* indicate the process to be followed in respect of the budget, intellectual property, and monitoring \n* use the [“SIB/Vital-IT DMP Canvas Generator tool”](https://dmp.vital-it.ch) to make your own DMP template. \n\n# Prerequisites\n##### ***Knowledge / competencies***\nTo be involved in Life Sciences research.\n\n##### ***Technical***\nYou will need a laptop with a web browser installed. \n\n\n# Trainers\n* Cécile Lebrand - Open Science advocate and information specialist, Bibliothèque Universitaire de Médecine, CHUV\n\n* Vassilios Ioannidis - SIB/Vital-IT Competence Center, SIB Swiss Institute of Bioinformatics\n\n* Grégoire Rossier - SIB/Vital-IT Competence Center & SIB Training, SIB Swiss Institute of Bioinformatics\n\n# Application\n\n\n\n\nThis course is free of charges. It is partially sponsored by [ELIXIR-CONVERGE](https://elixir-europe.org/about-us/how-funded/eu-projects/converge).\nWhile participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close as soon as the places will be filled up. Deadline for cancellation is set to **10/06/2022**. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. \n\n # Venue and time\n\n\nThis course will ONLY be held in person at the University of Lausanne (Metro M1 line, Sorge station). **No online streaming will be offered.**\n\nIt will start at 9:00 and end around 12:00 on the second day.\n\nPrecise information will be provided to the participants before the course.\n\n#  Additional information\nCoordination: Diana Marek, SIB Training Group\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).\n\n\n\n\n\n\n\n\nELIXIR-CONVERGE is funded by the European Commission within the Research Infrastructures programme of Horizon 2020\n\n\n\n","url":"https://www.sib.swiss/training2/website/course/20220623_XXX35","@id":"https://www.sib.swiss/training2/website/course/20220623_XXX35","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Research Data Management and Data Management Plan"}],"courseMode":["onsite","synchronous"],"keywords":"training,reproducibility,data management,data security,open data,mark ibberson group","location":"Lausanne","endDate":"2022-06-24","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Cécile Lebrand"},{"@type":"Person","name":"Grégoire Rossier"},{"@type":"Person","name":"Vassilios Ioannidis"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":0,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":0,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-06-23","duration":"P1DT6H"},{"@type":"CourseInstance","name":"Research Data Management and Data Management Plan","description":"\n\n\n\n\n\n\n\n# Overview\nRecent studies have shown that worldwide, between 51% and 89% of published life sciences research is not reproducible, with consequent losses estimated at around $100 billions/year in biomedical research (Chalmers *et al.*, 2009; Freedman *et al.*, 2015; Begley and Ioannidis, 2015). In particular, these studies have made clear that the research data associated with a publication are fundamental to validate the published analyses and results. Many causes contribute to this lack of reproducibility in life science studies such as a lack of rigor in data management and analysis. This extensive problem related to improper research management has urged scientists to consider developing efficient Research Data Management Strategies and implementing robust Data Management Plans (DMP) for their research projects. This need is also reflected in the requirements of funding agencies, amongst which the Swiss National Science Foundation (SNFS) and Horizon 2020.\n\n\nThis course, given by researchers and professionals involved in Big Data management at SIB/Vital-IT as well as in Data Management Plan preparation at UNIL/CHUV, will provide you with effective support to produce high quality DMP complying with the guidelines established by funding agencies. Importantly, it will provide you with tools to generate robust data and excellent quality studies that follow the FAIR principles to make your data Findable, Accessible, Interoperable and Reusable. \n\n*Sources of information*\n* Begley, C G, and Ioannidis, J. PA. “Reproducibility in science improving the standard for basic and preclinical research.” Circulation research. 2015; 116.1: 116-126. DOI: [10.1161/CIRCRESAHA.114.303819](https://doi.org/10.1161/circresaha.114.303819)\n* Chalmers I, Glasziou P. Avoidable Waste in the Production and Reporting of Research Evidence. Lancet. 2009; 374(9683): 86–89.  DOI: [10.1016/S0140-6736(09)60329-9](https://doi.org/10.1016/S0140-6736(09)60329-9)\n* Freedman LP, Cockburn IM, Simcoe TS. The Economics of Reproducibility in Preclinical Research. PLoS Biol. 2015;13(6): e1002165. DOI: [10.1371/journal.pbio.1002165](https://doi.org/10.1371/journal.pbio.1002165)\n\n# Schedule\n**Thursday 27 October 2022 - 09:00-17:00 CET**\n\nAt first, participants will be introduced to the notion of research reproducibility and to the need for a Data Management Plan (DMP) preparation, an evolving document reporting how the research data will be managed during and after a research project. You will learn best practices in Research Data Management (RDM) and especially two first steps concerning data collection and data documentation. During the exercises, you will directly apply what you have learned. \n\nDuring the afternoon, you will learn additional steps of the RDM cycle concerning Ethics, Legal, Security issues, Data Preservation and Data Sharing, as well as an overview of the FAIR principles. The remaining part of the course will be dedicated to demos and exercises on Data management, where you will learn how to share your published data on adapted repositories, such as Zenodo, and how to fill a DMP corresponding to your own research using the “SIB/Vital-IT DMP Canvas Generator tool”. \n\n\n**Friday 28 October 2022 - 09:00-12:30 CET**\n\nOn this half day, you will continue to complete your DPMs and refine the information under the guidance of the teachers. Those who would like will be able to present their draft DMPs in order to concretise their work and generate productive discussions in the group.\n\n# Audience\nThe course is addressed to postgraduate students and researchers who plan to apply for SNFS funds and want to be trained on how to efficiently complete the Data Management Plan form. At the same time, this course aims at educating the participants on Data Management principles in general.\n# Learning objectives\nAt the end of the course you will be able to:\n* put in place a DMP (data management plan) which fulfills the requirements of the funding agencies such as the SNFS and H2020, which require a DMP to be put in place\n* manage in detail your research data, specifying how your data will be analysed, organised, stored, secured and shared\n* specify the type of data that is going to be created and shared\n* indicate the process to be followed in respect of the budget, intellectual property, and monitoring \n* use the [“SIB/Vital-IT DMP Canvas Generator tool”](https://dmp.vital-it.ch) to make your own DMP template. \n\n# Prerequisites\n##### ***Knowledge / competencies***\nTo be involved in Life Sciences research.\n\n##### ***Technical***\nYou will need a laptop with a web browser installed.\n\n\n# Trainers\n\n* Cécile Lebrand - Open Science advocate and information specialist, Bibliothèque Universitaire de Médecine, CHUV\n\n* Vassilios Ioannidis - SIB/Vital-IT Competence Center, SIB Swiss Institute of Bioinformatics\n\n* Grégoire Rossier - SIB/Vital-IT Competence Center & SIB Training, SIB Swiss Institute of Bioinformatics\n\n# Application\n\n\nThis course is free-of-charge, as it is partially sponsored by [ELIXIR-CONVERGE](https://elixir-europe.org/about-us/how-funded/eu-projects/converge). However registration is mandatory (use the Apply button below). While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nYou will be informed by email of your registration confirmation. \n\nDeadline for cancellation is set to **13 October 2022**.  \n\n # Venue and time\n\n\n\n\nThis course will be held in person at the University of Bern.\n\nIt will start at 9:00 and end around 17:00.\n\nPrecise information will be provided to the participants before the course.\n\n#  Additional information\nCoordination: Monique Zahn, SIB Training Group.\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).\n\n\n\n\n\n\n\n\nELIXIR-CONVERGE is funded by the European Commission within the Research Infrastructures programme of Horizon 2020\n\n\n\n","url":"https://www.sib.swiss/training2/website/course/20221027_XXX35","@id":"https://www.sib.swiss/training2/website/course/20221027_XXX35","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Research Data Management and Data Management Plan"}],"courseMode":["onsite","synchronous"],"keywords":"data management,data security,open data,reproducibility,training,mark ibberson group","location":"Bern","endDate":"2022-10-28","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Cécile Lebrand"},{"@type":"Person","name":"Grégoire Rossier"},{"@type":"Person","name":"Vassilios Ioannidis"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":0,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":0,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-27","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"Single-cell Transcriptomics","description":"**The course is now full. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of upcoming events (https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses mailing list here (if not yet done): https://lists.sib.swiss/mailman/listinfo/courses. Thank you for your understanding.**\n\n\n# Overview\n\nIn contrast to bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq) allows researchers to study gene expression profiles at a single cell resolution. This enables the discovery of tissue specific cell subpopulations and markers, and has revolutionized the way we can use gene expression data to answer biological questions. This 3-day course will cover the most-used technologies to generate scRNA-seq data, as well as the main aspects to consider while designing a scRNAseq experiment. Participants will get hands-on experience in scRNA-seq data analysis by means of extensive  practical analysis sessions with R. The training materials for this course are in its dedicated GitHub page. [GitHub page](https://sib-swiss.github.io/single-cell-training).\n\n\n# Audience\nThis course is intended for life scientists familiar with next generation sequencing analysis methods who want to acquire the necessary skills to analyze scRNA-seq data.\n# Learning objectives\n\nAt the end of the course, participants will be able to:\n* Explain the characteristics of the most-used methods to generate transcriptomic data at the single-cell level, with a focus on 10x genomics. \n* Perform quality control at different steps of the analysis of single cell transcriptomic data. \n* Use dimensionality reduction (PCA, t-SNE, UMAP) and clustering algorithms for visualization and downstream analyses \n* Apply best practices for translating single cell transcriptomic data into biological knowledge by using cell annotation, differential gene expression analysis and trajectory analysis\n\n\n# Knowledge / competencies prerequired (Mandatory)\n\nParticipants should already have a basic knowledge in Next Generation Sequencing (NGS) techniques, or have already followed the \"[NGS - Quality control, Alignment, Visualisation](https://www.sib.swiss/training/course/20220516_NGSQCA)\". Knowledge in RNA sequencing is a plus. A basic knowledge of the R statistical software is required. [Test your R skills with the quiz here, before registering](https://docs.google.com/forms/d/e/1FAIpQLSdIyeuabd_ZOWXgI1MWHapmaOMu20L9ESkLDZiWnpmkpujyOg/viewform?usp=sf_link).\n\n# Technical requirements\n\nPlease check the required installation [detailed here](https://sib-swiss.github.io/single-cell-training/2021.11/precourse/ ), **before** the start of the course. \n\n# Program\n\nThe program of this course can be found in its dedicated [GitHub page](https://sib-swiss.github.io/single-cell-training/latest/course_schedule/).\n\n\n# Application\n\n\nThe registration fees for academics are **180 CHF** and **900 CHF** for for-profit companies.  While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed. \n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close once the places will be filled. Deadline for registration and free-of-charge cancellation is set to **20/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will take place at the University of Bern. \n\nIt will start at 9:15 and end around 17:15. \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Valeria Di Cola\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This course will cover the main technologies and aspects to consider while designing a scRNAseq experiment, and hands-on data analysis using R.","educationalLevel":"Intermediate","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Single-cell Transcriptomics","description":"**The course is now full. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of upcoming events (https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses mailing list here (if not yet done): https://lists.sib.swiss/mailman/listinfo/courses. Thank you for your understanding.**\n\n\n# Overview\n\nIn contrast to bulk RNA sequencing, single-cell RNA sequencing (scRNA-seq) allows researchers to study gene expression profiles at a single cell resolution. This enables the discovery of tissue specific cell subpopulations and markers, and has revolutionized the way we can use gene expression data to answer biological questions. This 3-day course will cover the most-used technologies to generate scRNA-seq data, as well as the main aspects to consider while designing a scRNAseq experiment. Participants will get hands-on experience in scRNA-seq data analysis by means of extensive  practical analysis sessions with R. The training materials for this course are in its dedicated GitHub page. [GitHub page](https://sib-swiss.github.io/single-cell-training).\n\n\n# Audience\nThis course is intended for life scientists familiar with next generation sequencing analysis methods who want to acquire the necessary skills to analyze scRNA-seq data.\n# Learning objectives\n\nAt the end of the course, participants will be able to:\n* Explain the characteristics of the most-used methods to generate transcriptomic data at the single-cell level, with a focus on 10x genomics. \n* Perform quality control at different steps of the analysis of single cell transcriptomic data. \n* Use dimensionality reduction (PCA, t-SNE, UMAP) and clustering algorithms for visualization and downstream analyses \n* Apply best practices for translating single cell transcriptomic data into biological knowledge by using cell annotation, differential gene expression analysis and trajectory analysis\n\n\n# Knowledge / competencies prerequired (Mandatory)\n\nParticipants should already have a basic knowledge in Next Generation Sequencing (NGS) techniques, or have already followed the \"[NGS - Quality control, Alignment, Visualisation](https://www.sib.swiss/training/course/20220516_NGSQCA)\". Knowledge in RNA sequencing is a plus. A basic knowledge of the R statistical software is required. [Test your R skills with the quiz here, before registering](https://docs.google.com/forms/d/e/1FAIpQLSdIyeuabd_ZOWXgI1MWHapmaOMu20L9ESkLDZiWnpmkpujyOg/viewform?usp=sf_link).\n\n# Technical requirements\n\nPlease check the required installation [detailed here](https://sib-swiss.github.io/single-cell-training/2021.11/precourse/ ), **before** the start of the course. \n\n# Program\n\nThe program of this course can be found in its dedicated [GitHub page](https://sib-swiss.github.io/single-cell-training/latest/course_schedule/).\n\n\n# Application\n\n\nThe registration fees for academics are **180 CHF** and **900 CHF** for for-profit companies.  While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed. \n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close once the places will be filled. Deadline for registration and free-of-charge cancellation is set to **20/06/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will take place at the University of Bern. \n\nIt will start at 9:15 and end around 17:15. \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Valeria Di Cola\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20220704_XXX36","@id":"https://www.sib.swiss/training2/website/course/20220704_XXX36","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Single-cell Transcriptomics"}],"courseMode":["onsite","synchronous"],"keywords":"genes and genomes,biochemistry,biomarkers,experimental biology,functional genomics,next generation sequencing,population genomics,single-cell biology,training,transcriptomics,mauro delorenzi & frédéric schütz group,rémy bruggmann group","location":"Bern","endDate":"2022-07-06","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Rachel Marcone"},{"@type":"Person","name":"Geert van Geest and Tania Wyss"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":180,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":900,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-07-04","duration":"P2DT5H"},{"@type":"CourseInstance","name":"Single-cell Transcriptomics","description":"\n\n**The course is now full with a long waiting list. If you do not want to miss your chance to be part of the next session and remain informed about all training activities at SIB, we highly recommend you to keep an eye on our list of upcoming events (https://www.sib.swiss/training/upcoming-training-courses) and subscribe to our courses mailing list here (if not yet done): https://lists.sib.swiss/mailman/listinfo/courses. Thank you for your understanding.**\n\n\n# Overview\n\nSingle-cell RNA sequencing (scRNAseq) allows researchers to study gene expression at the single cell level. For example, scRNAseq can help to identify expression patterns that differ between conditions within a cell-type. To generate and analyze scRNAseq data, several methods are available, all with their strengths and weaknesses depending on the researchers’ needs. This 3-day course will cover the main technologies as well as the main aspects to consider while designing a scRNAseq experiment. In addition, it will cover the theoretical background of analysis methods with hands-on practical data analysis sessions applied to droplet-based methods. All course material can be found on the course [web page](https://sib-swiss.github.io/single-cell-training/2022.7/). \n\n# Audience\nThis course is intended for life scientists and bioinformaticians familiar with \"Next Generation Sequencing\" who want to acquire the necessary skills to analyse scRNA-seq gene expression data.\n\n# Learning objectives\n\nAt the end of the course, participants will be able to:\n\n  * distinguish advantages and pitfalls of scRNAseq\n  * design their own scRNA-seq experiment\n  * apply a downstream analysis using R\n  \n\n# Knowledge / competencies prerequired (Mandatory)\n\nParticipants should already have a basic knowledge in Next Generation Sequencing (NGS) techniques, or have already followed the \"[NGS - Quality control, Alignment, Visualisation](https://www.sib.swiss/training/course/20220516_NGSQC)\". Knowledge in RNA sequencing is a plus. A basic knowledge of the R statistical software is required. [Test your R skills with the quiz here, before registering](https://docs.google.com/forms/d/e/1FAIpQLSdIyeuabd_ZOWXgI1MWHapmaOMu20L9ESkLDZiWnpmkpujyOg/viewform?usp=sf_link).\n\n\n# Technical requirements\nAttendees should have a Wi-Fi enabled computer. An online R and RStudio environment will be provided. However, in case you wish to perform the practical exercises on your own computer, please take a moment to install the following before the course:\n*  R version > 4.0.\n*  Latest RStudio version, the free version is perfectly fine.\n\n\n\n# Program\nProgram will be communicated in due time. Program of past occurences can be found on [GitHub](https://sib-swiss.github.io/single-cell-training/2022.7/course_schedule/).\n\n\n\n# Application\n\n\n**The course is now full with a long waiting list.**\n\nThe registration fees for academics are **180 CHF** and **900 CHF** for for-profit companies. While participants are registered on a first come, first served basis, exceptions may be made to ensure diversity and equity, which may increase the time before your registration is confirmed.\n\nApplications will close once the places will be filled. Deadline for registration and free-of-charge cancellation is set to **30/10/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\n# Venue and Time\n\nThis course will ONLY take place in person at the University of Bern\n\nIt will start at 9:15 CET and end around 17:00 CET every day.\n\nPlease note that a social dinner will be organised on 14 November after the course.\n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Diana Marek\n\nWe will recommend 0.75 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221114_XXX36","@id":"https://www.sib.swiss/training2/website/course/20221114_XXX36","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Single-cell Transcriptomics"}],"courseMode":["onsite","synchronous"],"keywords":"genes and genomes,biochemistry,biomarkers,experimental biology,functional genomics,next generation sequencing,population genomics,single-cell biology,training,transcriptomics,mauro delorenzi & frédéric schütz group,rémy bruggmann group","location":"Bern","endDate":"2022-11-16","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Rachel Marcone"},{"@type":"Person","name":"Geert van Geest and Tania Wyss"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":180,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":900,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-11-14","duration":"P2DT5H"}]},{"@context":"https://schema.org","@type":"Course","name":"Statistics and Machine Learning for Life Sciences","description":"# Overview\nStatistics are an integral aspect of scientific research, and in particular of life sciences that heavily rely on quantitative methodologies. Among other things, statistics are an essential tool which allows gaining new insights on the relationships between different biological measurements and variables. \n\nMachine learning (ML) also assists in making sense of large and complex datasets and can be very useful in mining large biological datasets to uncover new insights that can advance the field of bioinformatics.\n\nThis course was designed to guide participants in the exploration of the concepts of statistical modelling, and at the same time relate and contrast them with machine learning approaches when it comes to both classification and regression.\n\nA particular focus will be given on the evaluation of the relevance of the produced models, and their interpretation in order to provide new biological knowledge.\n\n# Audience\nThis course is addressed to life scientists who want to have a better understanding of these methods and on how to apply them to their own datasets. \n\n# Learning outcomes\nAt the end of the course, the participants will be able to:\n * perform linear and logistic regressions, and critically evaluate their results\n * describe the general Machine Learning data analysis pipeline\n * implement a classification task and appraise the resulting model\n * contrast the statistical and Machine Learning approaches when it comes to regression, and choose the most appropriate to their question.\n\n\n# Prerequisites\n##### Knowledge / competencies\nThe course is targeted to life scientists who are already familiar with the Python programming language and who have basic knowledge on statistics. The competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20220928_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20220620_STATP) and  [Introduction to statistics with R](https://www.sib.swiss/training/course/20220207_STATR).\n\n\nBefore applying to this course, please self assess your Python and statistics skills using the quiz [here.](https://forms.gle/ZpQFyHHwoPQKJSwv7) \n\n\n\n##### Technical\nYou are required to have your own computer with an internet connection and the following tools installed PRIOR to the course:\n* one of the latest Python 3 distributions (preferably above or equal to python 3.8), for instance installed with [conda](https://docs.continuum.io/anaconda/install/) \n* [Jupyter](https://jupyter.org/install)\n* the [scipy](https://www.scipy.org/install.html) library (NB: if you installed conda, then this library is already installed)\n* [scikitLearn](https://scikit-learn.org/stable/install.html) \n* [statsmodels](https://www.statsmodels.org/stable/install.html) \n\n\n\n# Schedule \n\nDay 1 \n* Warm-up: loading and plotting data with python. \n* Linear modelling: ordinary least squares, from fitting to models comparison\n* Logistic regression and Generalized Linear Models (GLM): from regression to classification\n\nDay 2 \n* The Machine Learning pipeline and evaluation\n* Machine Learning and classification: logistic regression classifier  and random forests\n* Machine Learning and regression\n\n# Application\n\n\n\nThe registration fees for academics are 120 CHF and 600 CHF for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the maximum capacity is reached. Deadline for free-of-charge cancellation is set to **24/11/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed from Basel.\n\nThe course will start at 9:00 and end around 17:00 (CET time zone). \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Valeria Di Cola, SIB training group\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":"This course is for life scientists to explore concepts of statistical modelling and machine learning, and gain new insights when mining large datasets.","educationalLevel":"Intermediate","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Statistics and Machine Learning for Life Sciences","description":"# Overview\nStatistics are an integral aspect of scientific research, and in particular of life sciences that heavily rely on quantitative methodologies. Among other things, statistics are an essential tool which allows gaining new insights on the relationships between different biological measurements and variables. \n\nMachine learning (ML) also assists in making sense of large and complex datasets and can be very useful in mining large biological datasets to uncover new insights that can advance the field of bioinformatics.\n\nThis course was designed to guide participants in the exploration of the concepts of statistical modelling, and at the same time relate and contrast them with machine learning approaches when it comes to both classification and regression.\n\nA particular focus will be given on the evaluation of the relevance of the produced models, and their interpretation in order to provide new biological knowledge.\n\n# Audience\nThis course is addressed to life scientists who want to have a better understanding of these methods and on how to apply them to their own datasets. \n\n# Learning outcomes\nAt the end of the course, the participants will be able to:\n * perform linear and logistic regressions, and critically evaluate their results\n * describe the general Machine Learning data analysis pipeline\n * implement a classification task and appraise the resulting model\n * contrast the statistical and Machine Learning approaches when it comes to regression, and choose the most appropriate to their question.\n\n\n# Prerequisites\n##### Knowledge / competencies\nThe course is targeted to life scientists who are already familiar with the Python programming language and who have basic knowledge on statistics. The competences and knowledge levels required correspond to those taught in courses such as: [First Steps with Python in Life Sciences](https://www.sib.swiss/training/course/20220928_PYTFS), [Introduction to statistics with Python](https://www.sib.swiss/training/course/20220620_STATP) and  [Introduction to statistics with R](https://www.sib.swiss/training/course/20220207_STATR).\n\n\nBefore applying to this course, please self assess your Python and statistics skills using the quiz [here.](https://forms.gle/ZpQFyHHwoPQKJSwv7) \n\n\n\n##### Technical\nYou are required to have your own computer with an internet connection and the following tools installed PRIOR to the course:\n* one of the latest Python 3 distributions (preferably above or equal to python 3.8), for instance installed with [conda](https://docs.continuum.io/anaconda/install/) \n* [Jupyter](https://jupyter.org/install)\n* the [scipy](https://www.scipy.org/install.html) library (NB: if you installed conda, then this library is already installed)\n* [scikitLearn](https://scikit-learn.org/stable/install.html) \n* [statsmodels](https://www.statsmodels.org/stable/install.html) \n\n\n\n# Schedule \n\nDay 1 \n* Warm-up: loading and plotting data with python. \n* Linear modelling: ordinary least squares, from fitting to models comparison\n* Logistic regression and Generalized Linear Models (GLM): from regression to classification\n\nDay 2 \n* The Machine Learning pipeline and evaluation\n* Machine Learning and classification: logistic regression classifier  and random forests\n* Machine Learning and regression\n\n# Application\n\n\n\nThe registration fees for academics are 120 CHF and 600 CHF for for-profit companies.\n\nYou will be informed by email of your registration confirmation. Upon reception of the confirmation email, participants will be asked to confirm attendance by paying the fees within 5 days.\n\nApplications will close as soon as the maximum capacity is reached. Deadline for free-of-charge cancellation is set to **24/11/2022**. Cancellation after this date will not be reimbursed. Please note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\n# Venue and Time\nThis course will be streamed from Basel.\n\nThe course will start at 9:00 and end around 17:00 (CET time zone). \n\nPrecise information will be provided to the participants in due time.\n\n\n#  Additional information\nCoordination: Valeria Di Cola, SIB training group\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nPlease note that participation in SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221208_XXX38","@id":"https://www.sib.swiss/training2/website/course/20221208_XXX38","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Statistics and Machine Learning for Life Sciences"}],"courseMode":["onsite","synchronous"],"keywords":"training,biostatistics,data mining,machine learning,torsten schwede & thierry sengstag group","location":"Streamed","endDate":"2022-12-09","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Wandrille Duchemin"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-12-08","duration":"P1DT6H"}]},{"@context":"https://schema.org","@type":"Course","name":"UNIX scripting for Life Scientists","description":"\n# Overview\nScripting allows shell users to automate repetitive tasks, thus improving reproducibility, reducing the risk of errors, freeing time, and avoiding boredom.\n\nBioinformatics analysis pipelines may involve dozens or hundreds of steps that are each carried out by different command-line programs: assembling these into scripts allows users to treat whole pipelines as if they were ordinary shell commands.\n\n# Audience\nThis course targets users who have basic knowledge of interactive shell use (such as taught in the SIB's [First Steps with UNIX for Life Scientist](https://www.sib.swiss/training/course/20220927_FSWU) and are interested in moving from interactive to automated tasks.\n\n# Learning objectives\nAt the end of the course, the participants should understand:\n* the main syntactic constructs of Bash (tests, conditionals, loops, functions)\n* how to read input and write output\n* how Bash stores and processes data (including the various kinds of expansion)\n* how to pass and parse command-line arguments and options, as well\n* how to to assemble individual analysis steps into reproducible, automated pipelines\n\n# Prerequisites\n##### Knowledge / competencies\nThis course is designed for intermediate level users. They are expected to know how to use a Unix shell interactively, i.e., moving around the filesystem, understanding pathnames, lauching programs that work on data, redirecting output to files or other programs, etc.\nThey are also expected to have some familiarity with basic shell utilities like grep, cut, tr, etc (although no expert skills are required).\nNotions of programming (in any language) will be useful, but not essential.\nFinally, some proficiency with a text editor is a must.\n\nMore details [here](https://sibcloud-my.sharepoint.com/:b:/g/personal/patricia_palagi_sib_swiss/EUg9dFU1_uBGm846H1n53J0BnZCrhaoq-P5ynt0oT5Yu8w?e=gJSVvu)\n\n\n\n##### Technical\nA laptop with a command line terminal and a recent version of Bash (4.0 or newer should be ok) installed. The alternative for Windows users is to activate [WSL/WSL2](https://learn.microsoft.com/en-us/windows/wsl/about).\n\n# Application\nRegistrations for this course are open.\n\nRegistration fees for academics are **120 CHF** and **600 CHF** for for-profit companies. \n\nDeadline for registration and free-of-charge cancellation is set to *27/10/2022*. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n\n# Venue and Time\nThe course will take place at the University of Bern, classroom information will be sent to the participants. It will start at 9:00 CET and end around 17:00 CET. The participants will get detailed information before the course.\n\n\n\n\n\n#  Additional information\nHelper: Robin Engler \nCoordination: Grégoire Rossier\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":null,"educationalLevel":"Intermediate","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"UNIX scripting for Life Scientists","description":"\n# Overview\nScripting allows shell users to automate repetitive tasks, thus improving reproducibility, reducing the risk of errors, freeing time, and avoiding boredom.\n\nBioinformatics analysis pipelines may involve dozens or hundreds of steps that are each carried out by different command-line programs: assembling these into scripts allows users to treat whole pipelines as if they were ordinary shell commands.\n\n# Audience\nThis course targets users who have basic knowledge of interactive shell use (such as taught in the SIB's [First Steps with UNIX for Life Scientist](https://www.sib.swiss/training/course/20220927_FSWU) and are interested in moving from interactive to automated tasks.\n\n# Learning objectives\nAt the end of the course, the participants should understand:\n* the main syntactic constructs of Bash (tests, conditionals, loops, functions)\n* how to read input and write output\n* how Bash stores and processes data (including the various kinds of expansion)\n* how to pass and parse command-line arguments and options, as well\n* how to to assemble individual analysis steps into reproducible, automated pipelines\n\n# Prerequisites\n##### Knowledge / competencies\nThis course is designed for intermediate level users. They are expected to know how to use a Unix shell interactively, i.e., moving around the filesystem, understanding pathnames, lauching programs that work on data, redirecting output to files or other programs, etc.\nThey are also expected to have some familiarity with basic shell utilities like grep, cut, tr, etc (although no expert skills are required).\nNotions of programming (in any language) will be useful, but not essential.\nFinally, some proficiency with a text editor is a must.\n\nMore details [here](https://sibcloud-my.sharepoint.com/:b:/g/personal/patricia_palagi_sib_swiss/EUg9dFU1_uBGm846H1n53J0BnZCrhaoq-P5ynt0oT5Yu8w?e=gJSVvu)\n\n\n\n##### Technical\nA laptop with a command line terminal and a recent version of Bash (4.0 or newer should be ok) installed. The alternative for Windows users is to activate [WSL/WSL2](https://learn.microsoft.com/en-us/windows/wsl/about).\n\n# Application\nRegistrations for this course are open.\n\nRegistration fees for academics are **120 CHF** and **600 CHF** for for-profit companies. \n\nDeadline for registration and free-of-charge cancellation is set to *27/10/2022*. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n\n# Venue and Time\nThe course will take place at the University of Bern, classroom information will be sent to the participants. It will start at 9:00 CET and end around 17:00 CET. The participants will get detailed information before the course.\n\n\n\n\n\n#  Additional information\nHelper: Robin Engler \nCoordination: Grégoire Rossier\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nSIB abides by the [ELIXIR Code of Conduct](https://elixir-europe.org/events/code-of-conduct). Participants of SIB courses are also required to abide by the same code.\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221107_XXX41","@id":"https://www.sib.swiss/training2/website/course/20221107_XXX41","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"UNIX scripting for Life Scientists"}],"courseMode":["onsite","synchronous"],"keywords":"programming","location":"Bern","endDate":"2022-11-08","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Thomas Junier"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-11-07"}]},{"@context":"https://schema.org","@type":"Course","name":"Version Control with Git","description":"\n\n\n# Overview\nGit is an open source, distributed, version control system for tracking changes in source code and\nother types of text documents. Created by Linus Torvald and first released in 2005, Git has become\nthe de-facto standard for project source code management, and is extensively used both in open\nsource and commercial software development. The usage of Git is not limited to code development,\nbut can also be used to keep track of data analysis scripts and pipelines.\nThis 2.5 day course gives a very comprehensive introduction to Git and its most useful commands, as well as an introduction to collaborative workflows and to using GitHub. The last half-day of the course is optional.\n\n# Audience\nThis course is aimed at people with no or little knowledge of Git, who are interested in using a version control system for collaborative work, or simply to keep track of modifications in their\nscripts/code base/files.\nThis includes people working on code development, but also scientists interested in improving the\nreproducibility of their data analyses by keeping track of their scripts using version control.\n\n# Learning objectives\nAt the end of the course, the participants are expected to have understood and practiced the following:\n\n* Introduction to Version Control Systems.\n* Basic Git concepts: commits, the HEAD pointer and the Git index.\n* Git branches: introduction to branched workflows and collaborative workflow examples.\n* Branch management: merge, rebase and cherry-pick.\n* Retrieving data from the Git database: git checkout.\n* Working with remotes: share your work and collaborate.\n* Basics of GitHub and GitLab.\n\n# Prerequisites\n### Knowledge/competencies:\nThe course is focused on using Git in command line mode (no graphical user interface). It is therefore necessary to have some basic knowledge of UNIX command line, e.g. how to change directory or how to edit a file in a command line editor such as vim/nano.\nIf you are not familiar with these UNIX fundamentals, we strongly recommend you to take the following [online tutorial](https://edu.sib.swiss/course/view.php?name=UNIX_I) before the start of the course.\n.\n\n### Technical\n* Your computer must be instllaed with a recent version of Git. Git is available on all major platforms (https://git-scm.com/download).\n* For online classes, you should also have a working microphone as some exercises involve\ncollaboration/communication with other participants.\n\n# Schedule\n### Day 1:\n* Introduction to version control systems and Git.\n* Git basics: your first commit.\n* Git concepts: commits, the HEAD pointer and the Git index.\n* Git branches: introduction to branched workflows and collaborative workflow examples.\n* Branch management: merge, rebase and cherry-pick.\n* Retrieving data from the Git database: git checkout.\n* Working with remotes: collaborating with Git.\n* GitHub: a brief overview.\n\n### Day 2:\n* Rewriting history: interactive rebase, git reset and commit amending.\n* The detached HEAD state explained.\n* The Git stash: Git’s “cut and paste” functionality.\n* Git tags: label important commits.\n* GitHub: creating a new project, adding new users and collaborating wit them.\n\n### Day 3 (half-day):\nThe last half-day of the course is optional and covers two Git extensions that can be useful in\ncertain scenarios:\n\n* Git LFS: versioning large files with Git.\n* Git submodules: embed a Git repository as a subdirectory of another Git repo.\n\n# Application\n\nRegistrations for this course are not yet open.\n\nRegistration fees for academics are **120 CHF**.\n\nDeadline for free-of-charge cancellation is set is set to *03/10/2022*. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n\n# Venue and Time\n\n\nUniversity of Lausanne\n\nOur courses usually start at 9:00 and end around 17:00. Precise information about location and time will be provided to the participants on due time.\n\n\n#  Additional information\nCoordination: Grégoire Rossier\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","inLanguage":"en-GB","provider":{"@type":"Organization","name":"SIB Swiss Institute of Bioinformatics","@id":"https://ror.org/002n09z45"},"abstract":null,"educationalLevel":"Beginner","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08","@type":"CreativeWork"},"hasCourseInstance":[{"@type":"CourseInstance","name":"Version Control with Git","description":"\n\n\n# Overview\nGit is an open source, distributed, version control system for tracking changes in source code and\nother types of text documents. Created by Linus Torvald and first released in 2005, Git has become\nthe de-facto standard for project source code management, and is extensively used both in open\nsource and commercial software development. The usage of Git is not limited to code development,\nbut can also be used to keep track of data analysis scripts and pipelines.\nThis 2.5 day course gives a very comprehensive introduction to Git and its most useful commands, as well as an introduction to collaborative workflows and to using GitHub. The last half-day of the course is optional.\n\n# Audience\nThis course is aimed at people with no or little knowledge of Git, who are interested in using a version control system for collaborative work, or simply to keep track of modifications in their\nscripts/code base/files.\nThis includes people working on code development, but also scientists interested in improving the\nreproducibility of their data analyses by keeping track of their scripts using version control.\n\n# Learning objectives\nAt the end of the course, the participants are expected to have understood and practiced the following:\n\n* Introduction to Version Control Systems.\n* Basic Git concepts: commits, the HEAD pointer and the Git index.\n* Git branches: introduction to branched workflows and collaborative workflow examples.\n* Branch management: merge, rebase and cherry-pick.\n* Retrieving data from the Git database: git checkout.\n* Working with remotes: share your work and collaborate.\n* Basics of GitHub and GitLab.\n\n# Prerequisites\n### Knowledge/competencies:\nThe course is focused on using Git in command line mode (no graphical user interface). It is therefore necessary to have some basic knowledge of UNIX command line, e.g. how to change directory or how to edit a file in a command line editor such as vim/nano.\nIf you are not familiar with these UNIX fundamentals, we strongly recommend you to take the following [online tutorial](https://edu.sib.swiss/course/view.php?name=UNIX_I) before the start of the course.\n.\n\n### Technical\n* Your computer must be instllaed with a recent version of Git. Git is available on all major platforms (https://git-scm.com/download).\n* For online classes, you should also have a working microphone as some exercises involve\ncollaboration/communication with other participants.\n\n# Schedule\n### Day 1:\n* Introduction to version control systems and Git.\n* Git basics: your first commit.\n* Git concepts: commits, the HEAD pointer and the Git index.\n* Git branches: introduction to branched workflows and collaborative workflow examples.\n* Branch management: merge, rebase and cherry-pick.\n* Retrieving data from the Git database: git checkout.\n* Working with remotes: collaborating with Git.\n* GitHub: a brief overview.\n\n### Day 2:\n* Rewriting history: interactive rebase, git reset and commit amending.\n* The detached HEAD state explained.\n* The Git stash: Git’s “cut and paste” functionality.\n* Git tags: label important commits.\n* GitHub: creating a new project, adding new users and collaborating wit them.\n\n### Day 3 (half-day):\nThe last half-day of the course is optional and covers two Git extensions that can be useful in\ncertain scenarios:\n\n* Git LFS: versioning large files with Git.\n* Git submodules: embed a Git repository as a subdirectory of another Git repo.\n\n# Application\n\nRegistrations for this course are not yet open.\n\nRegistration fees for academics are **120 CHF**.\n\nDeadline for free-of-charge cancellation is set is set to *03/10/2022*. Cancellation after this date will not be reimbursed. Please note that participation to SIB courses is subject to our [general conditions](http://www.sib.swiss/training/terms-and-conditions).\n\nYou will be informed by email of your registration confirmation.\n\n# Venue and Time\n\n\nUniversity of Lausanne\n\nOur courses usually start at 9:00 and end around 17:00. Precise information about location and time will be provided to the participants on due time.\n\n\n#  Additional information\nCoordination: Grégoire Rossier\n\nWe will recommend 0.5 ECTS credits for this course (given a passed exam at the end of the course).\n\nYou are welcome to register to the SIB courses mailing list to be informed of all future courses and workshops, as well as all important deadlines using the form [here](https://lists.sib.swiss/mailman/listinfo/courses).\n\nFor more information, please contact [training@sib.swiss](mailto://training@sib.swiss).","url":"https://www.sib.swiss/training2/website/course/20221012_XXX44","@id":"https://www.sib.swiss/training2/website/course/20221012_XXX44","http://purl.org/dc/terms/conformsTo":{"@id":"https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06","@type":"CreativeWork"},"about":[{"@type":"Event","name":"Version Control with Git"}],"courseMode":["onsite","synchronous"],"keywords":"programming,reproducibility,data management","location":"Lausanne","endDate":"2022-10-14","inLanguage":"en-GB","instructor":[{"@type":"Person","name":"Robin Engler"},{"@type":"Person","name":"Vassilios Ioannidis"}],"organizer":[{"@type":"Person","@id":"https://orcid.org/0000-0001-9062-6303","name":"Patricia Palagi"},{"@type":"Organization","@id":"https://ror.org/002n09z45","name":"SIB Swiss Institute of Bioinformatics"}],"offers":[{"@type":"Offer","name":"Academic Price","price":120,"eligibleCustomerType":{"@type":"BusinessEntityType","name":"Academic"},"priceCurrency":"CHF"},{"@type":"Offer","name":"Industry Price","price":600,"priceCurrency":"CHF","eligibleCustomerType":{"@type":"BusinessEntityType","name":"Industrial"}}],"startDate":"2022-10-12","duration":"P2DT5H"}]}]</script>
</head>

<body>

    
    <nav
    class="
    relative
    w-full
    flex flex-wrap
    items-center
    justify-between
    text-gray-500
    hover:text-gray-700
    focus:text-gray-700
    shadow-lg
    navbar navbar-expand-lg navbar-light
    ">
    <div class="container-fluid w-full flex flex-wrap items-start justify-between px-6">

        <div class="items-center">
            <a class="
          flex
          items-center
          text-gray-900
          hover:text-gray-900
          focus:text-gray-900
          mt-4
          mb-3
          mr-1
        "
                href="https://www.sib.swiss" target="sib">
                <img src="https://www.sib.swiss/training2/images/sib.png" alt="" loading="lazy" />
            </a>

        </div>
        <!-- Collapsible wrapper -->

        <!-- Right elements -->
        <div>
            <button
                class="
        navbar-toggler
        text-gray-500
        border-0
        hover:shadow-none hover:no-underline
        py-2
        px-2.5
        bg-transparent
        focus:outline-none focus:ring-0 focus:shadow-none focus:no-underline
      "
                type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
                aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="bars" class="w-6"
                    role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
                    <path fill="currentColor"
                        d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z">
                    </path>
                </svg>
            </button>
        </div>




        <div class="collapse navbar-collapse flex items-center relative" id="navbarSupportedContent">


            <div>
                <ul class="navbar-nav flex justify-end flex-col pl-0 list-style-none mr-auto">
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/careers" target="sib_website">
                            <span class="px-2">Careers</span></a>
                    </li>
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/contact" target="sib_website">
                            <span class="px-2">Contact</span></a>
                    </li>
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/directory" target="sib_website">
                            <span class="px-2">Directory</span></a>
                    </li>
                </ul>

                <ul class="navbar-nav flex flex-col pl-0 list-style-none mr-auto">
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200
                                    bg-gray-200 "
                            href="https://www.sib.swiss/training2/website/upcoming-training-courses">
                            <span class="px-2">
                                Training courses
                            </span>
                        </a>
                    </li>
                    
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/training/e-learning" target="sib_website">
                            <span class="px-2">e-learning</span></a>
                    </li>
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200
                                 "
                            href="https://www.sib.swiss/training2/website/training-materials">
                            <span class="px-2">Course materials</span></a>
                    </li>
                    
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/training/in-silico-talks" target="sib_website">
                            <span class="px-2">In silico talks</span></a>
                    </li>
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/training/who-can-benefit" target="sib_website">
                            <span class="px-2">Who can benefit</span></a>
                    </li>
                    <li class="nav-item mr-1">
                        <a class="nav-link py-4 inline-block text-gray-600  no-underline hover:bg-gray-200 focus:bg-gray-200"
                            href="https://www.sib.swiss/training/training-group" target="sib_website">
                            <span class="px-2">Training group</span></a>
                    </li>
                </ul>

            </div>
        </div>
        <!-- Right elements -->
    </div>
</nav>

</header>

    <div id="app">
            
    <upcoming-courses :app-url="'https://www.sib.swiss/training2'" :categories="[{&quot;id&quot;:1,&quot;name&quot;:&quot;Data management&quot;},{&quot;id&quot;:2,&quot;name&quot;:&quot;Machine Learning&quot;},{&quot;id&quot;:3,&quot;name&quot;:&quot;Omics data analysis&quot;},{&quot;id&quot;:4,&quot;name&quot;:&quot;Programming and Computing Techniques&quot;},{&quot;id&quot;:5,&quot;name&quot;:&quot;SIB resources&quot;},{&quot;id&quot;:6,&quot;name&quot;:&quot;Statistics&quot;}]" :levels="[&quot;Beginner&quot;,&quot;Intermediate&quot;,&quot;Advanced&quot;]"
        :cities="[&quot;Basel&quot;,&quot;Bern&quot;,&quot;Lausanne&quot;,&quot;Streamed&quot;,&quot;Zurich&quot;]" :statutes="[{&quot;value&quot;:&quot;upcoming&quot;,&quot;text&quot;:&quot;Upcoming&quot;,&quot;description&quot;:&quot;The course is not yet open to applications. Sign up to receive the notifications when a course opens for applications (usually 2 months before the start).&quot;},{&quot;value&quot;:&quot;open&quot;,&quot;text&quot;:&quot;Open&quot;,&quot;description&quot;:&quot;The course is open to applications.&quot;},{&quot;value&quot;:&quot;closed&quot;,&quot;text&quot;:&quot;Closed&quot;,&quot;description&quot;:&quot;The course is closed to applications. The course is either full with a long waiting list or has taken place.  Sign up to receive the notifications when a course opens for applications (usually 2 months before the start).&quot;},{&quot;value&quot;:&quot;cancelled&quot;,&quot;text&quot;:&quot;Cancelled&quot;,&quot;description&quot;:&quot;The course has been cancelled.  Sign up to receive the notifications when a course opens for applications (usually 2 months before the start).&quot;},{&quot;value&quot;:&quot;waiting-list&quot;,&quot;text&quot;:&quot;Waiting list&quot;,&quot;description&quot;:&quot;The course is open to applications, but you will be put on the waiting list.  Sign up to receive the notifications when a course opens for applications (usually 2 months before the start).&quot;}]"></upcoming-courses>


    <div class="container mx-auto text-center pt-12 pb-28">
        <a class="text-3xl no-underline" href="https://www.sib.swiss/training2/website/past-training-courses">Previous courses</a>
    </div>
    </div>

    <footer>
        <div class="container mx-auto">

            <div class="flex justify-between">

                <div class="w-1/3">
                    <h2>About SIB - <a href="https://www.sib.swiss" target="sib_website">https://www.sib.swiss</a></h2>
                    The SIB Swiss Institute of Bioinformatics is an internationally recognized non-profit organization,
                    dedicated to biological and biomedical data science. The institute contributes to keeping
                    Switzerland at the forefront of innovation by fostering progress in biological research and
                    enhancing health.
                    <br><br>
                    <a href="https://www.sib.swiss/about-sib/what-we-do">What we do »</a>
                </div>

                <div class="w-1/3">
                    <h2>About Training</h2>
                    The SIB Training courses ensure that bioinformaticians and life scientists make the best of the
                    latest advances in the field of computational biology.
                    <br><br>
                    <a href="https://www.sib.swiss/training/training-group">What we do »</a>
                </div>

                <div>
                    <h2>Contact</h2>

                    Quartier Sorge - Batiment Amphipole<br>
                    1015 Lausanne / Switzerland
                    <br><br>
                    Tel: +41 21 692 40 50
                    <br><br>
                    <a href="mailto:training@sib.swiss">Contact us »</a>
                </div>
            </div>

            <div class="footer-bottom">
                <h2>Partners</h2>
                <ul>
                    <li><a href="http://www.elixir-europe.org/" target="sib"> <img
                                src="https://www.sib.swiss/templates/sib/images/partners/logo_elixir.svg"
                                alt="Elixir Switzerland" style="width: 119px; height: 58px;"> </a></li>
                    <li><a href="https://www.fao.org/home/en" target="blank"> <img
                                src="https://www.sib.swiss/templates/sib/images/partners/logo_food_agricultur.png"
                                alt="Food and Agriculture Organization" class="d-none d-md-inline-block"></a></li>
                </ul>
            </div>

        </div>
    </footer>

    <button type="button"
        class="btn-back-to-top hidden fixed bg-white text-sib_blue bottom-12 right-8 rounded-full p-2 shadow-md opacity-60"
        title="Back to Top">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" aria-hidden="true" focusable="false"
            data-prefix="far" data-icon="arrow-alt-circle-up" class="w-6 h-6">
            <path fill="currentColor"
                d="M374.6 246.6C368.4 252.9 360.2 256 352 256s-16.38-3.125-22.62-9.375L224 141.3V448c0 17.69-14.33 31.1-31.1 31.1S160 465.7 160 448V141.3L54.63 246.6c-12.5 12.5-32.75 12.5-45.25 0s-12.5-32.75 0-45.25l160-160c12.5-12.5 32.75-12.5 45.25 0l160 160C387.1 213.9 387.1 234.1 374.6 246.6z" />
        </svg>
    </button>

    <script src="https://www.sib.swiss/training2/js/website.js?id=3b260463997e2ee774cdc227318c5a29" defer></script>
    </body>

</html>
